<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.33">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Brady Johnston">
<meta name="dcterms.date" content="2021-02-15">

<title>Interactively Visualising Proteins – Brady’s Blog</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-07ba0ad10f5680c660e360ac31d2f3b6.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting-dark-8b864f0777c60eecff11d75b6b2e1175.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap-cc1d910a108407c0aa672ecf2d09d0c9.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="../../site_libs/bootstrap/bootstrap-dark-1ca3e0674a0602a244ceba853400d2ef.min.css" rel="prefetch" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "show-item-context": false,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>
<script src="../../site_libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="../../site_libs/jquery-3.5.1/jquery.min.js"></script>
<script src="../../site_libs/r3dmol-1.6.2/3Dmol-nojquery-min.js"></script>
<script src="../../site_libs/r3dmol-binding-0.1.2/r3dmol.js"></script>


<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-fixed fullcontent">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top quarto-banner">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">Brady’s Blog</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/bradyajohnston"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://youtube.com/c/bradyjohnston"> <i class="bi bi-youtube" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/bradyajohnston"> <i class="bi bi-twitter" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../about.html"> 
<span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../index.html"> 
<span class="menu-text">Blog</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="https://bradyajohnston.github.io/MolecularNodes"> <i class="bi bi-arrow-up-short" role="img">
</i> 
<span class="menu-text">Molecular Nodes</span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
  <a href="" class="quarto-color-scheme-toggle quarto-navigation-tool  px-1" onclick="window.quartoToggleColorScheme(); return false;" title="Toggle dark mode"><i class="bi"></i></a>
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<header id="title-block-header" class="quarto-title-block default page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Interactively Visualising Proteins</h1>
            <p class="subtitle lead">My first publicly-available R package for structural bioinformatics.</p>
                                <div class="quarto-categories">
                <div class="quarto-category">bioinformatics</div>
                <div class="quarto-category">R</div>
                <div class="quarto-category">protein</div>
              </div>
                  </div>
  </div>
    
  
  <div class="quarto-title-meta">

      <div>
      <div class="quarto-title-meta-heading">Author</div>
      <div class="quarto-title-meta-contents">
               <p>Brady Johnston </p>
            </div>
    </div>
      
      <div>
      <div class="quarto-title-meta-heading">Published</div>
      <div class="quarto-title-meta-contents">
        <p class="date">February 15, 2021</p>
      </div>
    </div>
    
      
    </div>
    
  
  </header><div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    
<!-- main -->
<main class="content quarto-banner-title-block" id="quarto-document-content">





<section id="visualising-proteins" class="level1">
<h1>Visualising Proteins</h1>
<p>Recently I’ve been doing a lot of structural bioinformatics. To avoid having to export my results into PyMol of Chimerax, I wanted a better solution that could embed the results more easily into the report that I was writing.</p>
<p>What would be even better than just an image of the final result is a fully interactive 3D model of the protein I was analysing at the time. Packages to achieve a similar sort of impact were already available in the form of <a href="http://3dmol.csb.pitt.edu/">3Dmol.js</a>, but it wasn’t easy to embed or create through the R interface.</p>
<p>I now humbly present, an (early) version of <a href="https://swsoyee.github.io/r3dmol/articles/r3dmol.html">{r3dmol}</a>!</p>
<p>Play around with the widget below, which can be generated in just a couple of lines of code, and embedded in an <code>rmarkdown</code> document or in the RStudio viewer pane.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># install.packages("r3dmol")</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>r3dmol<span class="sc">::</span><span class="fu">m_glimpse</span>(<span class="st">'4ozs'</span>) <span class="sc">|&gt;</span> </span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>  r3dmol<span class="sc">::</span><span class="fu">m_spin</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="htmlwidget-6630f4ddf03de0feacfb" style="width:100%;height:500px;" class="r3dmol html-widget"></div>
<script type="application/json" data-for="htmlwidget-6630f4ddf03de0feacfb">{"x":{"id":"eb7ff874f89b2a115339","configs":{"nomouse":false,"backgroundColor":"white","cartoonQuality":10,"lowerZoomLimit":5,"upperZoomLimit":400,"antialias":true,"orthographic":false,"disableFog":false},"api":[{"method":"addModels","data":"HEADER    RNA BINDING PROTEIN                     18-FEB-14   4OZS              \nTITLE     RNA BINDING PROTEIN                                                   \nCOMPND    MOL_ID: 1;                                                            \nCOMPND   2 MOLECULE: ALPHA SOLENOID PROTEIN;                                    \nCOMPND   3 CHAIN: A;                                                            \nCOMPND   4 ENGINEERED: YES                                                      \nSOURCE    MOL_ID: 1;                                                            \nSOURCE   2 ORGANISM_SCIENTIFIC: ARTIFICIAL GENE;                                \nSOURCE   3 ORGANISM_TAXID: 32630;                                               \nSOURCE   4 EXPRESSION_SYSTEM: ESCHERICHIA COLI;                                 \nSOURCE   5 EXPRESSION_SYSTEM_TAXID: 562                                         \nKEYWDS    RNA, SPECIFICITY, RNA BINDING PROTEIN                                 \nEXPDTA    X-RAY DIFFRACTION                                                     \nAUTHOR    B.S.GULLY,C.S.BOND                                                    \nREVDAT   3   22-AUG-18 4OZS    1       REMARK LINK   ATOM                       \nREVDAT   2   22-NOV-17 4OZS    1       SOURCE REMARK                            \nREVDAT   1   22-APR-15 4OZS    0                                                \nJRNL        AUTH   B.S.GULLY,K.R.SHAH,M.LEE,K.SHEARSTON,N.M.SMITH,A.SADOWSKA,   \nJRNL        AUTH 2 A.J.BLYTHE,K.BERNATH-LEVIN,W.A.STANLEY,I.D.SMALL,C.S.BOND    \nJRNL        TITL   THE DESIGN AND STRUCTURAL CHARACTERIZATION OF A SYNTHETIC    \nJRNL        TITL 2 PENTATRICOPEPTIDE REPEAT PROTEIN.                            \nJRNL        REF    ACTA CRYSTALLOGR.,SECT.D      V.  71   196 2015              \nJRNL        REFN                   ESSN 1399-0047                               \nJRNL        PMID   25664731                                                     \nJRNL        DOI    10.1107/S1399004714024869                                    \nREMARK   2                                                                      \nREMARK   2 RESOLUTION.    2.17 ANGSTROMS.                                       \nREMARK   3                                                                      \nREMARK   3 REFINEMENT.                                                          \nREMARK   3   PROGRAM     : REFMAC 5.6.0117                                      \nREMARK   3   AUTHORS     : MURSHUDOV,SKUBAK,LEBEDEV,PANNU,STEINER,              \nREMARK   3               : NICHOLLS,WINN,LONG,VAGIN                             \nREMARK   3                                                                      \nREMARK   3    REFINEMENT TARGET : MAXIMUM LIKELIHOOD                            \nREMARK   3                                                                      \nREMARK   3  DATA USED IN REFINEMENT.                                            \nREMARK   3   RESOLUTION RANGE HIGH (ANGSTROMS) : 2.17                           \nREMARK   3   RESOLUTION RANGE LOW  (ANGSTROMS) : 56.28                          \nREMARK   3   DATA CUTOFF            (SIGMA(F)) : NULL                           \nREMARK   3   COMPLETENESS FOR RANGE        (%) : 99.8                           \nREMARK   3   NUMBER OF REFLECTIONS             : 17825                          \nREMARK   3                                                                      \nREMARK   3  FIT TO DATA USED IN REFINEMENT.                                     \nREMARK   3   CROSS-VALIDATION METHOD          : THROUGHOUT                      \nREMARK   3   FREE R VALUE TEST SET SELECTION  : RANDOM                          \nREMARK   3   R VALUE     (WORKING + TEST SET) : 0.219                           \nREMARK   3   R VALUE            (WORKING SET) : 0.217                           \nREMARK   3   FREE R VALUE                     : 0.265                           \nREMARK   3   FREE R VALUE TEST SET SIZE   (%) : 5.100                           \nREMARK   3   FREE R VALUE TEST SET COUNT      : 966                             \nREMARK   3                                                                      \nREMARK   3  FIT IN THE HIGHEST RESOLUTION BIN.                                  \nREMARK   3   TOTAL NUMBER OF BINS USED           : 20                           \nREMARK   3   BIN RESOLUTION RANGE HIGH       (A) : 2.17                         \nREMARK   3   BIN RESOLUTION RANGE LOW        (A) : 2.23                         \nREMARK   3   REFLECTION IN BIN     (WORKING SET) : 1155                         \nREMARK   3   BIN COMPLETENESS (WORKING+TEST) (%) : 97.67                        \nREMARK   3   BIN R VALUE           (WORKING SET) : 0.2890                       \nREMARK   3   BIN FREE R VALUE SET COUNT          : 59                           \nREMARK   3   BIN FREE R VALUE                    : 0.2890                       \nREMARK   3                                                                      \nREMARK   3  NUMBER OF NON-HYDROGEN ATOMS USED IN REFINEMENT.                    \nREMARK   3   PROTEIN ATOMS            : 1365                                    \nREMARK   3   NUCLEIC ACID ATOMS       : 0                                       \nREMARK   3   HETEROGEN ATOMS          : 0                                       \nREMARK   3   SOLVENT ATOMS            : 45                                      \nREMARK   3                                                                      \nREMARK   3  B VALUES.                                                           \nREMARK   3   FROM WILSON PLOT           (A**2) : NULL                           \nREMARK   3   MEAN B VALUE      (OVERALL, A**2) : 53.44                          \nREMARK   3   OVERALL ANISOTROPIC B VALUE.                                       \nREMARK   3    B11 (A**2) : 0.85000                                              \nREMARK   3    B22 (A**2) : -0.99000                                             \nREMARK   3    B33 (A**2) : 0.14000                                              \nREMARK   3    B12 (A**2) : 0.00000                                              \nREMARK   3    B13 (A**2) : 0.00000                                              \nREMARK   3    B23 (A**2) : 0.00000                                              \nREMARK   3                                                                      \nREMARK   3  ESTIMATED OVERALL COORDINATE ERROR.                                 \nREMARK   3   ESU BASED ON R VALUE                            (A): 0.164         \nREMARK   3   ESU BASED ON FREE R VALUE                       (A): 0.166         \nREMARK   3   ESU BASED ON MAXIMUM LIKELIHOOD                 (A): 0.127         \nREMARK   3   ESU FOR B VALUES BASED ON MAXIMUM LIKELIHOOD (A**2): 5.134         \nREMARK   3                                                                      \nREMARK   3 CORRELATION COEFFICIENTS.                                            \nREMARK   3   CORRELATION COEFFICIENT FO-FC      : 0.958                         \nREMARK   3   CORRELATION COEFFICIENT FO-FC FREE : 0.938                         \nREMARK   3                                                                      \nREMARK   3  RMS DEVIATIONS FROM IDEAL VALUES        COUNT    RMS    WEIGHT      \nREMARK   3   BOND LENGTHS REFINED ATOMS        (A):  1386 ; 0.021 ; 0.020       \nREMARK   3   BOND LENGTHS OTHERS               (A):  NULL ;  NULL ;  NULL       \nREMARK   3   BOND ANGLES REFINED ATOMS   (DEGREES):  1863 ; 2.442 ; 2.012       \nREMARK   3   BOND ANGLES OTHERS          (DEGREES):  NULL ;  NULL ;  NULL       \nREMARK   3   TORSION ANGLES, PERIOD 1    (DEGREES):   176 ; 6.539 ; 5.000       \nREMARK   3   TORSION ANGLES, PERIOD 2    (DEGREES):    61 ;44.576 ;26.721       \nREMARK   3   TORSION ANGLES, PERIOD 3    (DEGREES):   282 ;18.679 ;15.000       \nREMARK   3   TORSION ANGLES, PERIOD 4    (DEGREES):     5 ; 7.426 ;15.000       \nREMARK   3   CHIRAL-CENTER RESTRAINTS       (A**3):   216 ; 0.154 ; 0.200       \nREMARK   3   GENERAL PLANES REFINED ATOMS      (A):  1004 ; 0.010 ; 0.021       \nREMARK   3   GENERAL PLANES OTHERS             (A):  NULL ;  NULL ;  NULL       \nREMARK   3   NON-BONDED CONTACTS REFINED ATOMS (A):  NULL ;  NULL ;  NULL       \nREMARK   3   NON-BONDED CONTACTS OTHERS        (A):  NULL ;  NULL ;  NULL       \nREMARK   3   NON-BONDED TORSION REFINED ATOMS  (A):  NULL ;  NULL ;  NULL       \nREMARK   3   NON-BONDED TORSION OTHERS         (A):  NULL ;  NULL ;  NULL       \nREMARK   3   H-BOND (X...Y) REFINED ATOMS      (A):  NULL ;  NULL ;  NULL       \nREMARK   3   H-BOND (X...Y) OTHERS             (A):  NULL ;  NULL ;  NULL       \nREMARK   3   POTENTIAL METAL-ION REFINED ATOMS (A):  NULL ;  NULL ;  NULL       \nREMARK   3   POTENTIAL METAL-ION OTHERS        (A):  NULL ;  NULL ;  NULL       \nREMARK   3   SYMMETRY VDW REFINED ATOMS        (A):  NULL ;  NULL ;  NULL       \nREMARK   3   SYMMETRY VDW OTHERS               (A):  NULL ;  NULL ;  NULL       \nREMARK   3   SYMMETRY H-BOND REFINED ATOMS     (A):  NULL ;  NULL ;  NULL       \nREMARK   3   SYMMETRY H-BOND OTHERS            (A):  NULL ;  NULL ;  NULL       \nREMARK   3   SYMMETRY METAL-ION REFINED ATOMS  (A):  NULL ;  NULL ;  NULL       \nREMARK   3   SYMMETRY METAL-ION OTHERS         (A):  NULL ;  NULL ;  NULL       \nREMARK   3                                                                      \nREMARK   3  ISOTROPIC THERMAL FACTOR RESTRAINTS.     COUNT   RMS    WEIGHT      \nREMARK   3   MAIN-CHAIN BOND REFINED ATOMS  (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   MAIN-CHAIN BOND OTHER ATOMS    (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   MAIN-CHAIN ANGLE REFINED ATOMS (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   MAIN-CHAIN ANGLE OTHER ATOMS   (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   SIDE-CHAIN BOND REFINED ATOMS  (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   SIDE-CHAIN BOND OTHER ATOMS    (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   SIDE-CHAIN ANGLE REFINED ATOMS (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   SIDE-CHAIN ANGLE OTHER ATOMS   (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   LONG RANGE B REFINED ATOMS     (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   LONG RANGE B OTHER ATOMS       (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3                                                                      \nREMARK   3 ANISOTROPIC THERMAL FACTOR RESTRAINTS.    COUNT   RMS   WEIGHT       \nREMARK   3   RIGID-BOND RESTRAINTS          (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   SPHERICITY; FREE ATOMS         (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3   SPHERICITY; BONDED ATOMS       (A**2):  NULL ;  NULL ;  NULL       \nREMARK   3                                                                      \nREMARK   3  NCS RESTRAINTS STATISTICS                                           \nREMARK   3   NUMBER OF DIFFERENT NCS GROUPS : NULL                              \nREMARK   3                                                                      \nREMARK   3  TLS DETAILS                                                         \nREMARK   3   NUMBER OF TLS GROUPS  : NULL                                       \nREMARK   3                                                                      \nREMARK   3  BULK SOLVENT MODELLING.                                             \nREMARK   3   METHOD USED : MASK                                                 \nREMARK   3   PARAMETERS FOR MASK CALCULATION                                    \nREMARK   3   VDW PROBE RADIUS   : 1.20                                          \nREMARK   3   ION PROBE RADIUS   : 0.80                                          \nREMARK   3   SHRINKAGE RADIUS   : 0.80                                          \nREMARK   3                                                                      \nREMARK   3  OTHER REFINEMENT REMARKS: HYDROGENS HAVE BEEN USED IF PRESENT IN    \nREMARK   3  THE INPUT                                                           \nREMARK   4                                                                      \nREMARK   4 4OZS COMPLIES WITH FORMAT V. 3.30, 13-JUL-11                         \nREMARK 100                                                                      \nREMARK 100 THIS ENTRY HAS BEEN PROCESSED BY RCSB ON 11-MAR-14.                  \nREMARK 100 THE DEPOSITION ID IS D_1000200111.                                   \nREMARK 200                                                                      \nREMARK 200 EXPERIMENTAL DETAILS                                                 \nREMARK 200  EXPERIMENT TYPE                : X-RAY DIFFRACTION                  \nREMARK 200  DATE OF DATA COLLECTION        : 05-JUL-13                          \nREMARK 200  TEMPERATURE           (KELVIN) : 100                                \nREMARK 200  PH                             : 3.35                               \nREMARK 200  NUMBER OF CRYSTALS USED        : 1                                  \nREMARK 200                                                                      \nREMARK 200  SYNCHROTRON              (Y/N) : Y                                  \nREMARK 200  RADIATION SOURCE               : AUSTRALIAN SYNCHROTRON             \nREMARK 200  BEAMLINE                       : MX2                                \nREMARK 200  X-RAY GENERATOR MODEL          : NULL                               \nREMARK 200  MONOCHROMATIC OR LAUE    (M/L) : NULL                               \nREMARK 200  WAVELENGTH OR RANGE        (A) : 0.980, 0.954                       \nREMARK 200  MONOCHROMATOR                  : NULL                               \nREMARK 200  OPTICS                         : NULL                               \nREMARK 200                                                                      \nREMARK 200  DETECTOR TYPE                  : CCD                                \nREMARK 200  DETECTOR MANUFACTURER          : ADSC QUANTUM 315R                  \nREMARK 200  INTENSITY-INTEGRATION SOFTWARE : XSCALE                             \nREMARK 200  DATA SCALING SOFTWARE          : XDS, SCALA                         \nREMARK 200                                                                      \nREMARK 200  NUMBER OF UNIQUE REFLECTIONS   : 18791                              \nREMARK 200  RESOLUTION RANGE HIGH      (A) : 2.170                              \nREMARK 200  RESOLUTION RANGE LOW       (A) : 56.280                             \nREMARK 200  REJECTION CRITERIA  (SIGMA(I)) : NULL                               \nREMARK 200                                                                      \nREMARK 200 OVERALL.                                                             \nREMARK 200  COMPLETENESS FOR RANGE     (%) : 99.8                               \nREMARK 200  DATA REDUNDANCY                : 2.000                              \nREMARK 200  R MERGE                    (I) : 0.01873                            \nREMARK 200  R SYM                      (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR THE DATA SET  : 24.4400                            \nREMARK 200                                                                      \nREMARK 200 IN THE HIGHEST RESOLUTION SHELL.                                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE HIGH (A) : 2.17                     \nREMARK 200  HIGHEST RESOLUTION SHELL, RANGE LOW  (A) : 2.25                     \nREMARK 200  COMPLETENESS FOR SHELL     (%) : 98.4                               \nREMARK 200  DATA REDUNDANCY IN SHELL       : 2.00                               \nREMARK 200  R MERGE FOR SHELL          (I) : 0.31340                            \nREMARK 200  R SYM FOR SHELL            (I) : NULL                               \nREMARK 200  <I/SIGMA(I)> FOR SHELL         : 2.680                              \nREMARK 200                                                                      \nREMARK 200 DIFFRACTION PROTOCOL: MAD                                            \nREMARK 200 METHOD USED TO DETERMINE THE STRUCTURE: NULL                         \nREMARK 200 SOFTWARE USED: PHENIX                                                \nREMARK 200 STARTING MODEL: NULL                                                 \nREMARK 200                                                                      \nREMARK 200 REMARK: NULL                                                         \nREMARK 280                                                                      \nREMARK 280 CRYSTAL                                                              \nREMARK 280 SOLVENT CONTENT, VS   (%): 72.22                                     \nREMARK 280 MATTHEWS COEFFICIENT, VM (ANGSTROMS**3/DA): 4.43                     \nREMARK 280                                                                      \nREMARK 280 CRYSTALLIZATION CONDITIONS: DIFFRACTION QUALITY CRYSTALS GREW FROM   \nREMARK 280  3 UL DROPS IN 24-WELL SITTING DROP CRYSCHEM PLATES (HAMPTON         \nREMARK 280  RESEARCH) IN 2:1 RATIOS OF CRYSTALLANT 100 MM SODIUM CITRATE PH     \nREMARK 280  3.35, 8 % (W/V) PEG 3350 AND PROTEIN EQUILIBRATED AGAINST 1 ML      \nREMARK 280  OF CRYSTALLANT AT 293 K., VAPOR DIFFUSION, HANGING DROP             \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY                                            \nREMARK 290 SYMMETRY OPERATORS FOR SPACE GROUP: P 21 21 21                       \nREMARK 290                                                                      \nREMARK 290      SYMOP   SYMMETRY                                                \nREMARK 290     NNNMMM   OPERATOR                                                \nREMARK 290       1555   X,Y,Z                                                   \nREMARK 290       2555   -X+1/2,-Y,Z+1/2                                         \nREMARK 290       3555   -X,Y+1/2,-Z+1/2                                         \nREMARK 290       4555   X+1/2,-Y+1/2,-Z                                         \nREMARK 290                                                                      \nREMARK 290     WHERE NNN -> OPERATOR NUMBER                                     \nREMARK 290           MMM -> TRANSLATION VECTOR                                  \nREMARK 290                                                                      \nREMARK 290 CRYSTALLOGRAPHIC SYMMETRY TRANSFORMATIONS                            \nREMARK 290 THE FOLLOWING TRANSFORMATIONS OPERATE ON THE ATOM/HETATM             \nREMARK 290 RECORDS IN THIS ENTRY TO PRODUCE CRYSTALLOGRAPHICALLY                \nREMARK 290 RELATED MOLECULES.                                                   \nREMARK 290   SMTRY1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 290   SMTRY1   2 -1.000000  0.000000  0.000000       27.01250            \nREMARK 290   SMTRY2   2  0.000000 -1.000000  0.000000        0.00000            \nREMARK 290   SMTRY3   2  0.000000  0.000000  1.000000       42.55850            \nREMARK 290   SMTRY1   3 -1.000000  0.000000  0.000000        0.00000            \nREMARK 290   SMTRY2   3  0.000000  1.000000  0.000000       37.51150            \nREMARK 290   SMTRY3   3  0.000000  0.000000 -1.000000       42.55850            \nREMARK 290   SMTRY1   4  1.000000  0.000000  0.000000       27.01250            \nREMARK 290   SMTRY2   4  0.000000 -1.000000  0.000000       37.51150            \nREMARK 290   SMTRY3   4  0.000000  0.000000 -1.000000        0.00000            \nREMARK 290                                                                      \nREMARK 290 REMARK: NULL                                                         \nREMARK 300                                                                      \nREMARK 300 BIOMOLECULE: 1                                                       \nREMARK 300 SEE REMARK 350 FOR THE AUTHOR PROVIDED AND/OR PROGRAM                \nREMARK 300 GENERATED ASSEMBLY INFORMATION FOR THE STRUCTURE IN                  \nREMARK 300 THIS ENTRY. THE REMARK MAY ALSO PROVIDE INFORMATION ON               \nREMARK 300 BURIED SURFACE AREA.                                                 \nREMARK 350                                                                      \nREMARK 350 COORDINATES FOR A COMPLETE MULTIMER REPRESENTING THE KNOWN           \nREMARK 350 BIOLOGICALLY SIGNIFICANT OLIGOMERIZATION STATE OF THE                \nREMARK 350 MOLECULE CAN BE GENERATED BY APPLYING BIOMT TRANSFORMATIONS          \nREMARK 350 GIVEN BELOW.  BOTH NON-CRYSTALLOGRAPHIC AND                          \nREMARK 350 CRYSTALLOGRAPHIC OPERATIONS ARE GIVEN.                               \nREMARK 350                                                                      \nREMARK 350 BIOMOLECULE: 1                                                       \nREMARK 350 AUTHOR DETERMINED BIOLOGICAL UNIT: TRIMERIC                          \nREMARK 350 APPLY THE FOLLOWING TO CHAINS: A                                     \nREMARK 350   BIOMT1   1  1.000000  0.000000  0.000000        0.00000            \nREMARK 350   BIOMT2   1  0.000000  1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   1  0.000000  0.000000  1.000000        0.00000            \nREMARK 350   BIOMT1   2 -1.000000  0.000000  0.000000       81.03750            \nREMARK 350   BIOMT2   2  0.000000 -1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   2  0.000000  0.000000  1.000000       42.55850            \nREMARK 350   BIOMT1   3 -1.000000  0.000000  0.000000       81.03750            \nREMARK 350   BIOMT2   3  0.000000 -1.000000  0.000000        0.00000            \nREMARK 350   BIOMT3   3  0.000000  0.000000  1.000000      -42.55850            \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: CLOSE CONTACTS IN SAME ASYMMETRIC UNIT                     \nREMARK 500                                                                      \nREMARK 500 THE FOLLOWING ATOMS ARE IN CLOSE CONTACT.                            \nREMARK 500                                                                      \nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI           DISTANCE          \nREMARK 500   O    HOH A   231     O    HOH A   243              2.11            \nREMARK 500   OE2  GLU A    55     O    HOH A   201              2.11            \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: CLOSE CONTACTS                                             \nREMARK 500                                                                      \nREMARK 500 THE FOLLOWING ATOMS THAT ARE RELATED BY CRYSTALLOGRAPHIC             \nREMARK 500 SYMMETRY ARE IN CLOSE CONTACT.  AN ATOM LOCATED WITHIN 0.15          \nREMARK 500 ANGSTROMS OF A SYMMETRY RELATED ATOM IS ASSUMED TO BE ON A           \nREMARK 500 SPECIAL POSITION AND IS, THEREFORE, LISTED IN REMARK 375             \nREMARK 500 INSTEAD OF REMARK 500.  ATOMS WITH NON-BLANK ALTERNATE               \nREMARK 500 LOCATION INDICATORS ARE NOT INCLUDED IN THE CALCULATIONS.            \nREMARK 500                                                                      \nREMARK 500 DISTANCE CUTOFF:                                                     \nREMARK 500 2.2 ANGSTROMS FOR CONTACTS NOT INVOLVING HYDROGEN ATOMS              \nREMARK 500 1.6 ANGSTROMS FOR CONTACTS INVOLVING HYDROGEN ATOMS                  \nREMARK 500                                                                      \nREMARK 500  ATM1  RES C  SSEQI   ATM2  RES C  SSEQI  SSYMOP   DISTANCE          \nREMARK 500   N    ASP A     1     O    PRO A   175     2654     2.14            \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND LENGTHS                                      \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,2(A3,1X,A1,I4,A1,1X,A4,3X),1X,F6.3)               \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   RES CSSEQI ATM2   DEVIATION                     \nREMARK 500    GLU A  90   CD    GLU A  90   OE1     0.068                       \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: COVALENT BOND ANGLES                                       \nREMARK 500                                                                      \nREMARK 500 THE STEREOCHEMICAL PARAMETERS OF THE FOLLOWING RESIDUES              \nREMARK 500 HAVE VALUES WHICH DEVIATE FROM EXPECTED VALUES BY MORE               \nREMARK 500 THAN 6*RMSD (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN               \nREMARK 500 IDENTIFIER; SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                 \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT: (10X,I3,1X,A3,1X,A1,I4,A1,3(1X,A4,2X),12X,F5.1)              \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES PROTEIN: ENGH AND HUBER, 1999                        \nREMARK 500 EXPECTED VALUES NUCLEIC ACID: CLOWNEY ET AL 1996                     \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI ATM1   ATM2   ATM3                                     \nREMARK 500    MET A  28   CG  -  SD  -  CE  ANGL. DEV. = -11.8 DEGREES          \nREMARK 500    MET A  63   CG  -  SD  -  CE  ANGL. DEV. =  -9.7 DEGREES          \nREMARK 500    MET A  98   CG  -  SD  -  CE  ANGL. DEV. =  -9.6 DEGREES          \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nREMARK 500                                                                      \nREMARK 500 GEOMETRY AND STEREOCHEMISTRY                                         \nREMARK 500 SUBTOPIC: TORSION ANGLES                                             \nREMARK 500                                                                      \nREMARK 500 TORSION ANGLES OUTSIDE THE EXPECTED RAMACHANDRAN REGIONS:            \nREMARK 500 (M=MODEL NUMBER; RES=RESIDUE NAME; C=CHAIN IDENTIFIER;               \nREMARK 500 SSEQ=SEQUENCE NUMBER; I=INSERTION CODE).                             \nREMARK 500                                                                      \nREMARK 500 STANDARD TABLE:                                                      \nREMARK 500 FORMAT:(10X,I3,1X,A3,1X,A1,I4,A1,4X,F7.2,3X,F7.2)                    \nREMARK 500                                                                      \nREMARK 500 EXPECTED VALUES: GJ KLEYWEGT AND TA JONES (1996). PHI/PSI-           \nREMARK 500 CHOLOGY: RAMACHANDRAN REVISITED. STRUCTURE 4, 1395 - 1400            \nREMARK 500                                                                      \nREMARK 500  M RES CSSEQI        PSI       PHI                                   \nREMARK 500    ASP A  36     -167.18   -122.47                                   \nREMARK 500    PRO A 140     -172.57    -57.57                                   \nREMARK 500    THR A 144      -75.84    -62.32                                   \nREMARK 500    TYR A 145      -57.70    -23.70                                   \nREMARK 500                                                                      \nREMARK 500 REMARK: NULL                                                         \nDBREF  4OZS A    1   175  PDB    4OZS     4OZS             1    175             \nSEQRES   1 A  175  ASP VAL VAL THR TYR ASN THR LEU ILE ASP GLY LEU ALA          \nSEQRES   2 A  175  LYS ALA GLY ARG LEU GLU GLU ALA LEU GLN LEU PHE GLN          \nSEQRES   3 A  175  GLU MET LYS GLU LYS GLY VAL LYS PRO ASP VAL VAL THR          \nSEQRES   4 A  175  TYR ASN THR LEU ILE ASP GLY LEU ALA LYS ALA GLY ARG          \nSEQRES   5 A  175  LEU GLU GLU ALA LEU GLN LEU PHE GLN GLU MET LYS GLU          \nSEQRES   6 A  175  LYS GLY VAL LYS PRO ASP VAL VAL THR TYR ASN THR LEU          \nSEQRES   7 A  175  ILE ASP GLY LEU ALA LYS ALA GLY ARG LEU GLU GLU ALA          \nSEQRES   8 A  175  LEU GLN LEU PHE GLN GLU MET LYS GLU LYS GLY VAL LYS          \nSEQRES   9 A  175  PRO ASP VAL VAL THR TYR ASN THR LEU ILE ASP GLY LEU          \nSEQRES  10 A  175  ALA LYS ALA GLY ARG LEU GLU GLU ALA LEU GLN LEU PHE          \nSEQRES  11 A  175  GLN GLU MET LYS GLU LYS GLY VAL LYS PRO ASP VAL VAL          \nSEQRES  12 A  175  THR TYR ASN THR LEU ILE ASP GLY LEU ALA LYS ALA GLY          \nSEQRES  13 A  175  ARG LEU GLU GLU ALA LEU GLN LEU PHE GLN GLU MET LYS          \nSEQRES  14 A  175  GLU LYS GLY VAL LYS PRO                                      \nFORMUL   2  HOH   *45(H2 O)                                                     \nHELIX    1 AA1 ASP A    1  ALA A   15  1                                  15    \nHELIX    2 AA2 ARG A   17  LYS A   31  1                                  15    \nHELIX    3 AA3 ASP A   36  ALA A   50  1                                  15    \nHELIX    4 AA4 ARG A   52  LYS A   66  1                                  15    \nHELIX    5 AA5 ASP A   71  ALA A   85  1                                  15    \nHELIX    6 AA6 ARG A   87  LYS A  101  1                                  15    \nHELIX    7 AA7 ASP A  106  ALA A  120  1                                  15    \nHELIX    8 AA8 ARG A  122  LYS A  136  1                                  15    \nHELIX    9 AA9 ASP A  141  ALA A  155  1                                  15    \nHELIX   10 AB1 ARG A  157  LYS A  171  1                                  15    \nLINK         N   ASP A   1                 C   PRO A 175     1555   2654  1.74  \nCRYST1   54.025   75.023   85.117  90.00  90.00  90.00 P 21 21 21    4          \nORIGX1      1.000000  0.000000  0.000000        0.00000                         \nORIGX2      0.000000  1.000000  0.000000        0.00000                         \nORIGX3      0.000000  0.000000  1.000000        0.00000                         \nSCALE1      0.018510  0.000000  0.000000        0.00000                         \nSCALE2      0.000000  0.013329  0.000000        0.00000                         \nSCALE3      0.000000  0.000000  0.011749        0.00000                         \nATOM      1  N   ASP A   1      42.099  12.935 -17.274  1.00 43.88           N  \nATOM      2  CA  ASP A   1      40.682  12.606 -17.121  1.00 55.80           C  \nATOM      3  C   ASP A   1      40.605  12.058 -15.711  1.00 57.97           C  \nATOM      4  O   ASP A   1      41.631  11.916 -15.024  1.00 64.28           O  \nATOM      5  CB  ASP A   1      40.272  11.534 -18.141  1.00 63.10           C  \nATOM      6  CG  ASP A   1      41.162  10.287 -18.062  1.00 59.15           C  \nATOM      7  OD1 ASP A   1      41.296   9.672 -16.981  1.00 64.73           O  \nATOM      8  OD2 ASP A   1      41.775   9.965 -19.091  1.00 76.58           O  \nATOM      9  N   VAL A   2      39.394  11.714 -15.302  1.00 59.57           N  \nATOM     10  CA  VAL A   2      39.137  11.285 -13.940  1.00 60.76           C  \nATOM     11  C   VAL A   2      40.013  10.135 -13.499  1.00 53.51           C  \nATOM     12  O   VAL A   2      40.359  10.021 -12.305  1.00 53.02           O  \nATOM     13  CB  VAL A   2      37.667  10.922 -13.757  1.00 59.21           C  \nATOM     14  CG1 VAL A   2      37.299  10.908 -12.273  1.00 59.51           C  \nATOM     15  CG2 VAL A   2      36.852  11.938 -14.512  1.00 58.27           C  \nATOM     16  N   VAL A   3      40.396   9.286 -14.439  1.00 52.28           N  \nATOM     17  CA  VAL A   3      41.223   8.144 -14.025  1.00 56.71           C  \nATOM     18  C   VAL A   3      42.682   8.590 -13.642  1.00 45.14           C  \nATOM     19  O   VAL A   3      43.190   8.139 -12.626  1.00 44.69           O  \nATOM     20  CB  VAL A   3      41.241   6.978 -15.059  1.00 65.62           C  \nATOM     21  CG1 VAL A   3      41.965   5.767 -14.468  1.00 55.57           C  \nATOM     22  CG2 VAL A   3      39.826   6.583 -15.493  1.00 58.79           C  \nATOM     23  N   THR A   4      43.308   9.478 -14.430  1.00 43.84           N  \nATOM     24  CA  THR A   4      44.610  10.054 -14.057  1.00 47.49           C  \nATOM     25  C   THR A   4      44.502  10.677 -12.642  1.00 42.35           C  \nATOM     26  O   THR A   4      45.236  10.326 -11.715  1.00 45.78           O  \nATOM     27  CB  THR A   4      45.126  11.208 -14.991  1.00 44.33           C  \nATOM     28  OG1 THR A   4      44.618  11.131 -16.332  1.00 52.34           O  \nATOM     29  CG2 THR A   4      46.629  11.217 -15.029  1.00 38.52           C  \nATOM     30  N   TYR A   5      43.539  11.569 -12.475  1.00 39.20           N  \nATOM     31  CA  TYR A   5      43.407  12.259 -11.195  1.00 38.94           C  \nATOM     32  C   TYR A   5      43.135  11.343 -10.059  1.00 38.73           C  \nATOM     33  O   TYR A   5      43.831  11.422  -8.989  1.00 39.57           O  \nATOM     34  CB  TYR A   5      42.340  13.352 -11.300  1.00 34.90           C  \nATOM     35  CG  TYR A   5      42.901  14.561 -11.995  1.00 38.84           C  \nATOM     36  CD1 TYR A   5      43.831  15.398 -11.325  1.00 39.02           C  \nATOM     37  CD2 TYR A   5      42.514  14.908 -13.301  1.00 42.01           C  \nATOM     38  CE1 TYR A   5      44.353  16.535 -11.943  1.00 37.55           C  \nATOM     39  CE2 TYR A   5      43.058  16.030 -13.936  1.00 43.60           C  \nATOM     40  CZ  TYR A   5      43.955  16.846 -13.238  1.00 41.28           C  \nATOM     41  OH  TYR A   5      44.525  17.957 -13.827  1.00 48.31           O  \nATOM     42  N   ASN A   6      42.192  10.408 -10.258  1.00 39.47           N  \nATOM     43  CA  ASN A   6      41.836   9.482  -9.121  1.00 40.53           C  \nATOM     44  C   ASN A   6      42.999   8.625  -8.675  1.00 34.39           C  \nATOM     45  O   ASN A   6      43.284   8.397  -7.479  1.00 38.59           O  \nATOM     46  CB  ASN A   6      40.639   8.592  -9.499  1.00 47.28           C  \nATOM     47  CG  ASN A   6      39.271   9.326  -9.379  1.00 53.80           C  \nATOM     48  OD1 ASN A   6      39.185  10.522  -9.092  1.00 44.92           O  \nATOM     49  ND2 ASN A   6      38.213   8.599  -9.636  1.00 59.00           N  \nATOM     50  N   THR A   7      43.752   8.158  -9.647  1.00 40.74           N  \nATOM     51  CA  THR A   7      44.926   7.342  -9.237  1.00 44.91           C  \nATOM     52  C   THR A   7      45.941   8.157  -8.452  1.00 37.90           C  \nATOM     53  O   THR A   7      46.474   7.709  -7.426  1.00 41.37           O  \nATOM     54  CB  THR A   7      45.671   6.862 -10.483  1.00 55.24           C  \nATOM     55  OG1 THR A   7      44.711   6.526 -11.490  1.00 51.23           O  \nATOM     56  CG2 THR A   7      46.559   5.687 -10.124  1.00 53.61           C  \nATOM     57  N   LEU A   8      46.223   9.384  -8.909  1.00 42.13           N  \nATOM     58  CA  LEU A   8      47.273  10.190  -8.165  1.00 34.66           C  \nATOM     59  C   LEU A   8      46.729  10.560  -6.822  1.00 30.66           C  \nATOM     60  O   LEU A   8      47.398  10.468  -5.766  1.00 31.96           O  \nATOM     61  CB  LEU A   8      47.574  11.421  -9.002  1.00 32.92           C  \nATOM     62  CG  LEU A   8      48.372  10.980 -10.212  1.00 35.74           C  \nATOM     63  CD1 LEU A   8      48.595  12.142 -11.167  1.00 34.59           C  \nATOM     64  CD2 LEU A   8      49.737  10.479  -9.779  1.00 39.55           C  \nATOM     65  N   ILE A   9      45.465  10.981  -6.826  1.00 35.35           N  \nATOM     66  CA  ILE A   9      44.870  11.495  -5.562  1.00 40.24           C  \nATOM     67  C   ILE A   9      44.832  10.297  -4.597  1.00 41.92           C  \nATOM     68  O   ILE A   9      45.202  10.372  -3.395  1.00 40.73           O  \nATOM     69  CB  ILE A   9      43.454  12.088  -5.803  1.00 40.53           C  \nATOM     70  CG1 ILE A   9      43.596  13.402  -6.614  1.00 39.99           C  \nATOM     71  CG2 ILE A   9      42.768  12.282  -4.450  1.00 41.14           C  \nATOM     72  CD1 ILE A   9      42.285  13.951  -7.185  1.00 43.98           C  \nATOM     73  N   ASP A  10      44.477   9.162  -5.148  1.00 43.95           N  \nATOM     74  CA  ASP A  10      44.389   7.968  -4.267  1.00 49.14           C  \nATOM     75  C   ASP A  10      45.731   7.560  -3.749  1.00 42.38           C  \nATOM     76  O   ASP A  10      45.929   7.377  -2.527  1.00 47.70           O  \nATOM     77  CB  ASP A  10      43.742   6.769  -4.954  1.00 58.16           C  \nATOM     78  CG  ASP A  10      43.850   5.519  -4.071  1.00 77.86           C  \nATOM     79  OD1 ASP A  10      43.192   5.466  -3.013  1.00 77.81           O  \nATOM     80  OD2 ASP A  10      44.664   4.629  -4.369  1.00 81.53           O  \nATOM     81  N   GLY A  11      46.721   7.511  -4.643  1.00 40.89           N  \nATOM     82  CA  GLY A  11      48.061   7.218  -4.127  1.00 41.36           C  \nATOM     83  C   GLY A  11      48.539   8.253  -3.115  1.00 44.78           C  \nATOM     84  O   GLY A  11      49.090   7.903  -2.062  1.00 48.49           O  \nATOM     85  N   LEU A  12      48.367   9.551  -3.407  1.00 43.31           N  \nATOM     86  CA  LEU A  12      48.870  10.604  -2.427  1.00 38.55           C  \nATOM     87  C   LEU A  12      48.140  10.468  -1.104  1.00 39.15           C  \nATOM     88  O   LEU A  12      48.729  10.662  -0.019  1.00 39.26           O  \nATOM     89  CB  LEU A  12      48.610  12.043  -2.959  1.00 37.28           C  \nATOM     90  CG  LEU A  12      49.366  12.418  -4.239  1.00 43.21           C  \nATOM     91  CD1 LEU A  12      48.660  13.571  -4.938  1.00 37.76           C  \nATOM     92  CD2 LEU A  12      50.755  12.844  -3.766  1.00 40.16           C  \nATOM     93  N   ALA A  13      46.825  10.210  -1.189  1.00 41.15           N  \nATOM     94  CA  ALA A  13      46.044  10.112   0.080  1.00 45.38           C  \nATOM     95  C   ALA A  13      46.578   8.930   0.890  1.00 52.27           C  \nATOM     96  O   ALA A  13      46.871   9.088   2.054  1.00 54.97           O  \nATOM     97  CB  ALA A  13      44.546   9.970  -0.198  1.00 46.58           C  \nATOM     98  N   LYS A  14      46.805   7.784   0.244  1.00 55.95           N  \nATOM     99  CA  LYS A  14      47.392   6.635   0.932  1.00 61.34           C  \nATOM    100  C   LYS A  14      48.708   6.982   1.587  1.00 61.85           C  \nATOM    101  O   LYS A  14      48.987   6.484   2.659  1.00 57.75           O  \nATOM    102  CB  LYS A  14      47.676   5.489  -0.028  1.00 62.42           C  \nATOM    103  CG  LYS A  14      46.492   4.636  -0.400  1.00 71.38           C  \nATOM    104  CD  LYS A  14      47.071   3.295  -0.848  1.00 80.82           C  \nATOM    105  CE  LYS A  14      46.159   2.566  -1.814  1.00 82.85           C  \nATOM    106  NZ  LYS A  14      44.729   2.710  -1.416  1.00 86.13           N  \nATOM    107  N   ALA A  15      49.544   7.800   0.952  1.00 57.69           N  \nATOM    108  CA  ALA A  15      50.819   8.132   1.573  1.00 49.24           C  \nATOM    109  C   ALA A  15      50.670   9.199   2.651  1.00 53.36           C  \nATOM    110  O   ALA A  15      51.640   9.652   3.212  1.00 63.98           O  \nATOM    111  CB  ALA A  15      51.833   8.548   0.546  1.00 51.93           C  \nATOM    112  N   GLY A  16      49.452   9.616   2.945  1.00 52.21           N  \nATOM    113  CA  GLY A  16      49.244  10.728   3.891  1.00 51.81           C  \nATOM    114  C   GLY A  16      49.482  12.126   3.330  1.00 53.06           C  \nATOM    115  O   GLY A  16      49.413  13.058   4.072  1.00 47.47           O  \nATOM    116  N   ARG A  17      49.705  12.261   2.020  1.00 46.84           N  \nATOM    117  CA  ARG A  17      50.124  13.521   1.392  1.00 48.27           C  \nATOM    118  C   ARG A  17      48.873  14.208   0.925  1.00 45.11           C  \nATOM    119  O   ARG A  17      48.578  14.322  -0.278  1.00 44.60           O  \nATOM    120  CB  ARG A  17      51.085  13.219   0.233  1.00 48.36           C  \nATOM    121  CG  ARG A  17      52.362  12.534   0.704  1.00 43.29           C  \nATOM    122  CD  ARG A  17      53.305  12.073  -0.425  1.00 43.84           C  \nATOM    123  NE  ARG A  17      53.569  13.139  -1.419  1.00 48.01           N  \nATOM    124  CZ  ARG A  17      54.450  13.048  -2.418  1.00 43.19           C  \nATOM    125  NH1 ARG A  17      55.153  11.950  -2.583  1.00 39.08           N  \nATOM    126  NH2 ARG A  17      54.604  14.033  -3.290  1.00 37.12           N  \nATOM    127  N   LEU A  18      48.108  14.644   1.916  1.00 44.31           N  \nATOM    128  CA  LEU A  18      46.764  15.134   1.706  1.00 44.65           C  \nATOM    129  C   LEU A  18      46.730  16.527   1.151  1.00 41.88           C  \nATOM    130  O   LEU A  18      45.770  16.873   0.461  1.00 46.69           O  \nATOM    131  CB  LEU A  18      45.978  15.105   3.051  1.00 49.25           C  \nATOM    132  CG  LEU A  18      45.354  13.757   3.368  1.00 49.00           C  \nATOM    133  CD1 LEU A  18      44.913  13.947   4.810  1.00 57.81           C  \nATOM    134  CD2 LEU A  18      44.153  13.594   2.399  1.00 46.59           C  \nATOM    135  N   GLU A  19      47.741  17.338   1.462  1.00 40.02           N  \nATOM    136  CA  GLU A  19      47.775  18.692   0.883  1.00 44.29           C  \nATOM    137  C   GLU A  19      47.925  18.589  -0.651  1.00 36.32           C  \nATOM    138  O   GLU A  19      47.199  19.247  -1.396  1.00 41.58           O  \nATOM    139  CB  GLU A  19      48.894  19.554   1.461  1.00 45.90           C  \nATOM    140  CG  GLU A  19      48.722  19.974   2.916  1.00 52.59           C  \nATOM    141  CD  GLU A  19      47.354  20.551   3.197  1.00 62.20           C  \nATOM    142  OE1 GLU A  19      46.935  21.488   2.504  1.00 64.39           O  \nATOM    143  OE2 GLU A  19      46.675  20.023   4.101  1.00 71.65           O  \nATOM    144  N   GLU A  20      48.869  17.806  -1.111  1.00 31.47           N  \nATOM    145  CA AGLU A  20      49.014  17.538  -2.551  0.50 35.66           C  \nATOM    146  CA BGLU A  20      48.956  17.638  -2.551  0.50 34.68           C  \nATOM    147  C   GLU A  20      47.746  16.883  -3.154  1.00 37.68           C  \nATOM    148  O   GLU A  20      47.311  17.210  -4.285  1.00 36.73           O  \nATOM    149  CB AGLU A  20      50.200  16.612  -2.761  0.50 34.23           C  \nATOM    150  CB BGLU A  20      50.267  17.023  -2.963  0.50 32.79           C  \nATOM    151  CG AGLU A  20      51.529  17.238  -2.419  0.50 36.94           C  \nATOM    152  CG BGLU A  20      50.429  17.111  -4.463  0.50 33.75           C  \nATOM    153  CD AGLU A  20      52.637  16.207  -2.235  0.50 36.86           C  \nATOM    154  CD BGLU A  20      51.816  16.756  -4.891  0.50 31.59           C  \nATOM    155  OE1AGLU A  20      52.905  15.808  -1.096  0.50 28.64           O  \nATOM    156  OE1BGLU A  20      52.641  16.536  -4.009  0.50 27.95           O  \nATOM    157  OE2AGLU A  20      53.224  15.791  -3.263  0.50 38.91           O  \nATOM    158  OE2BGLU A  20      52.065  16.679  -6.108  0.50 29.34           O  \nATOM    159  N   ALA A  21      47.157  15.923  -2.414  1.00 36.92           N  \nATOM    160  CA  ALA A  21      45.951  15.229  -2.953  1.00 38.14           C  \nATOM    161  C   ALA A  21      44.805  16.224  -3.123  1.00 32.43           C  \nATOM    162  O   ALA A  21      44.121  16.231  -4.149  1.00 31.57           O  \nATOM    163  CB  ALA A  21      45.509  14.055  -2.052  1.00 35.22           C  \nATOM    164  N   LEU A  22      44.661  17.115  -2.144  1.00 36.28           N  \nATOM    165  CA  LEU A  22      43.609  18.119  -2.214  1.00 37.18           C  \nATOM    166  C   LEU A  22      43.906  19.131  -3.266  1.00 38.47           C  \nATOM    167  O   LEU A  22      43.017  19.659  -3.885  1.00 40.74           O  \nATOM    168  CB  LEU A  22      43.456  18.815  -0.873  1.00 41.10           C  \nATOM    169  CG  LEU A  22      42.842  18.051   0.335  1.00 47.19           C  \nATOM    170  CD1 LEU A  22      43.021  18.869   1.615  1.00 44.07           C  \nATOM    171  CD2 LEU A  22      41.356  17.890   0.072  1.00 40.37           C  \nATOM    172  N   GLN A  23      45.180  19.383  -3.519  1.00 39.05           N  \nATOM    173  CA  GLN A  23      45.508  20.380  -4.553  1.00 37.47           C  \nATOM    174  C   GLN A  23      45.143  19.798  -5.901  1.00 35.48           C  \nATOM    175  O   GLN A  23      44.595  20.482  -6.795  1.00 38.27           O  \nATOM    176  CB  GLN A  23      47.020  20.714  -4.439  1.00 40.96           C  \nATOM    177  CG  GLN A  23      47.520  21.780  -5.413  1.00 48.40           C  \nATOM    178  CD  GLN A  23      46.732  23.108  -5.264  1.00 53.87           C  \nATOM    179  OE1 GLN A  23      46.746  23.705  -4.198  1.00 57.75           O  \nATOM    180  NE2 GLN A  23      46.013  23.538  -6.322  1.00 45.65           N  \nATOM    181  N   LEU A  24      45.462  18.508  -6.090  1.00 33.22           N  \nATOM    182  CA  LEU A  24      45.102  17.892  -7.404  1.00 37.35           C  \nATOM    183  C   LEU A  24      43.630  17.809  -7.590  1.00 29.86           C  \nATOM    184  O   LEU A  24      43.120  18.013  -8.667  1.00 36.78           O  \nATOM    185  CB  LEU A  24      45.650  16.455  -7.537  1.00 34.21           C  \nATOM    186  CG  LEU A  24      47.135  16.394  -7.796  1.00 43.00           C  \nATOM    187  CD1 LEU A  24      47.363  15.011  -8.352  1.00 38.69           C  \nATOM    188  CD2 LEU A  24      47.454  17.500  -8.844  1.00 42.39           C  \nATOM    189  N   PHE A  25      42.940  17.575  -6.480  1.00 36.58           N  \nATOM    190  CA  PHE A  25      41.487  17.693  -6.471  1.00 34.02           C  \nATOM    191  C   PHE A  25      40.943  19.027  -6.993  1.00 38.32           C  \nATOM    192  O   PHE A  25      40.037  19.083  -7.858  1.00 34.20           O  \nATOM    193  CB  PHE A  25      41.036  17.464  -5.066  1.00 41.21           C  \nATOM    194  CG  PHE A  25      39.522  17.474  -4.885  1.00 46.08           C  \nATOM    195  CD1 PHE A  25      38.701  16.535  -5.541  1.00 40.77           C  \nATOM    196  CD2 PHE A  25      38.925  18.402  -4.026  1.00 45.12           C  \nATOM    197  CE1 PHE A  25      37.310  16.546  -5.376  1.00 42.23           C  \nATOM    198  CE2 PHE A  25      37.512  18.382  -3.834  1.00 53.27           C  \nATOM    199  CZ  PHE A  25      36.711  17.438  -4.504  1.00 43.41           C  \nATOM    200  N   GLN A  26      41.467  20.109  -6.426  1.00 41.81           N  \nATOM    201  CA  GLN A  26      41.088  21.417  -6.840  1.00 41.21           C  \nATOM    202  C   GLN A  26      41.450  21.615  -8.320  1.00 41.97           C  \nATOM    203  O   GLN A  26      40.625  22.094  -9.131  1.00 42.55           O  \nATOM    204  CB  GLN A  26      41.851  22.436  -5.925  1.00 48.81           C  \nATOM    205  CG  GLN A  26      41.601  23.888  -6.284  1.00 55.43           C  \nATOM    206  CD  GLN A  26      40.105  24.151  -6.454  1.00 71.80           C  \nATOM    207  OE1 GLN A  26      39.356  24.095  -5.474  1.00 68.53           O  \nATOM    208  NE2 GLN A  26      39.653  24.382  -7.707  1.00 80.93           N  \nATOM    209  N   GLU A  27      42.688  21.237  -8.684  1.00 35.30           N  \nATOM    210  CA  GLU A  27      43.103  21.353 -10.087  1.00 40.02           C  \nATOM    211  C   GLU A  27      42.145  20.649 -11.059  1.00 44.83           C  \nATOM    212  O   GLU A  27      41.821  21.153 -12.158  1.00 42.49           O  \nATOM    213  CB  GLU A  27      44.496  20.735 -10.258  1.00 37.05           C  \nATOM    214  CG  GLU A  27      45.041  20.926 -11.653  1.00 42.88           C  \nATOM    215  CD  GLU A  27      46.404  20.298 -11.789  1.00 43.46           C  \nATOM    216  OE1 GLU A  27      47.239  20.472 -10.858  1.00 46.18           O  \nATOM    217  OE2 GLU A  27      46.581  19.558 -12.790  1.00 49.26           O  \nATOM    218  N   MET A  28      41.752  19.446 -10.688  1.00 44.12           N  \nATOM    219  CA  MET A  28      40.799  18.629 -11.497  1.00 50.49           C  \nATOM    220  C   MET A  28      39.522  19.417 -11.848  1.00 48.26           C  \nATOM    221  O   MET A  28      39.039  19.442 -13.008  1.00 45.37           O  \nATOM    222  CB  MET A  28      40.332  17.529 -10.560  1.00 52.42           C  \nATOM    223  CG  MET A  28      40.056  16.213 -11.187  1.00 54.98           C  \nATOM    224  SD  MET A  28      39.604  15.209  -9.749  1.00 52.27           S  \nATOM    225  CE  MET A  28      38.494  14.271 -10.818  1.00 49.87           C  \nATOM    226  N   LYS A  29      38.979  20.066 -10.818  1.00 44.98           N  \nATOM    227  CA  LYS A  29      37.674  20.747 -10.944  1.00 51.05           C  \nATOM    228  C   LYS A  29      37.840  21.923 -11.867  1.00 53.41           C  \nATOM    229  O   LYS A  29      37.046  22.088 -12.794  1.00 59.35           O  \nATOM    230  CB  LYS A  29      37.177  21.234  -9.590  1.00 42.71           C  \nATOM    231  CG  LYS A  29      36.654  20.110  -8.746  1.00 48.38           C  \nATOM    232  CD  LYS A  29      36.859  20.549  -7.318  1.00 52.32           C  \nATOM    233  CE  LYS A  29      35.753  20.139  -6.408  1.00 50.56           C  \nATOM    234  NZ  LYS A  29      36.150  20.915  -5.172  1.00 50.77           N  \nATOM    235  N   GLU A  30      38.917  22.684 -11.651  1.00 53.86           N  \nATOM    236  CA  GLU A  30      39.258  23.842 -12.473  1.00 54.56           C  \nATOM    237  C   GLU A  30      39.408  23.519 -13.926  1.00 59.86           C  \nATOM    238  O   GLU A  30      39.109  24.351 -14.733  1.00 65.60           O  \nATOM    239  CB  GLU A  30      40.580  24.438 -12.045  1.00 57.92           C  \nATOM    240  CG  GLU A  30      40.491  25.399 -10.892  1.00 66.19           C  \nATOM    241  CD  GLU A  30      41.821  25.563 -10.186  1.00 89.09           C  \nATOM    242  OE1 GLU A  30      42.916  25.300 -10.782  1.00 89.51           O  \nATOM    243  OE2 GLU A  30      41.762  25.934  -8.998  1.00100.85           O  \nATOM    244  N   LYS A  31      39.933  22.348 -14.275  1.00 58.62           N  \nATOM    245  CA  LYS A  31      40.027  21.977 -15.679  1.00 50.18           C  \nATOM    246  C   LYS A  31      38.693  21.456 -16.173  1.00 56.92           C  \nATOM    247  O   LYS A  31      38.595  20.973 -17.292  1.00 62.78           O  \nATOM    248  CB  LYS A  31      41.028  20.837 -15.895  1.00 58.40           C  \nATOM    249  CG  LYS A  31      42.352  20.954 -15.201  1.00 59.27           C  \nATOM    250  CD  LYS A  31      43.126  22.098 -15.741  1.00 56.62           C  \nATOM    251  CE  LYS A  31      44.571  21.774 -15.558  1.00 66.60           C  \nATOM    252  NZ  LYS A  31      45.209  22.820 -16.389  1.00 74.45           N  \nATOM    253  N   GLY A  32      37.669  21.451 -15.338  1.00 60.11           N  \nATOM    254  CA  GLY A  32      36.418  20.870 -15.799  1.00 59.54           C  \nATOM    255  C   GLY A  32      36.522  19.365 -15.959  1.00 65.12           C  \nATOM    256  O   GLY A  32      35.686  18.756 -16.613  1.00 70.16           O  \nATOM    257  N   VAL A  33      37.524  18.736 -15.354  1.00 62.51           N  \nATOM    258  CA  VAL A  33      37.477  17.252 -15.201  1.00 63.93           C  \nATOM    259  C   VAL A  33      36.656  16.877 -13.938  1.00 65.35           C  \nATOM    260  O   VAL A  33      37.024  17.265 -12.826  1.00 63.31           O  \nATOM    261  CB  VAL A  33      38.907  16.654 -15.109  1.00 61.47           C  \nATOM    262  CG1 VAL A  33      38.846  15.175 -14.762  1.00 59.76           C  \nATOM    263  CG2 VAL A  33      39.673  16.884 -16.413  1.00 57.01           C  \nATOM    264  N   LYS A  34      35.568  16.122 -14.094  1.00 73.76           N  \nATOM    265  CA  LYS A  34      34.574  15.923 -13.008  1.00 65.92           C  \nATOM    266  C   LYS A  34      34.999  14.946 -11.873  1.00 63.96           C  \nATOM    267  O   LYS A  34      35.163  13.741 -12.139  1.00 70.68           O  \nATOM    268  CB  LYS A  34      33.275  15.401 -13.625  1.00 74.48           C  \nATOM    269  CG  LYS A  34      32.024  16.056 -13.105  1.00 80.38           C  \nATOM    270  CD  LYS A  34      31.546  17.093 -14.097  1.00 88.80           C  \nATOM    271  CE  LYS A  34      30.953  18.275 -13.356  1.00 96.04           C  \nATOM    272  NZ  LYS A  34      31.701  19.506 -13.728  1.00 84.90           N  \nATOM    273  N   PRO A  35      35.161  15.437 -10.598  1.00 61.11           N  \nATOM    274  CA  PRO A  35      35.352  14.466  -9.472  1.00 59.89           C  \nATOM    275  C   PRO A  35      34.186  13.406  -9.412  1.00 58.47           C  \nATOM    276  O   PRO A  35      33.040  13.776  -9.710  1.00 59.27           O  \nATOM    277  CB  PRO A  35      35.308  15.382  -8.233  1.00 53.06           C  \nATOM    278  CG  PRO A  35      35.809  16.715  -8.748  1.00 53.07           C  \nATOM    279  CD  PRO A  35      35.116  16.837 -10.087  1.00 50.78           C  \nATOM    280  N   ASP A  36      34.469  12.131  -9.082  1.00 59.65           N  \nATOM    281  CA  ASP A  36      33.402  11.132  -8.822  1.00 59.75           C  \nATOM    282  C   ASP A  36      33.503  10.505  -7.432  1.00 68.07           C  \nATOM    283  O   ASP A  36      34.274  10.973  -6.565  1.00 63.23           O  \nATOM    284  CB  ASP A  36      33.315  10.041  -9.914  1.00 55.15           C  \nATOM    285  CG  ASP A  36      34.509   9.101  -9.914  1.00 61.88           C  \nATOM    286  OD1 ASP A  36      35.078   8.740  -8.854  1.00 61.07           O  \nATOM    287  OD2 ASP A  36      34.873   8.688 -11.028  1.00 75.87           O  \nATOM    288  N   VAL A  37      32.743   9.418  -7.242  1.00 71.01           N  \nATOM    289  CA  VAL A  37      32.463   8.846  -5.924  1.00 62.22           C  \nATOM    290  C   VAL A  37      33.766   8.495  -5.250  1.00 61.11           C  \nATOM    291  O   VAL A  37      34.019   8.718  -4.030  1.00 56.49           O  \nATOM    292  CB  VAL A  37      31.589   7.555  -6.086  1.00 67.39           C  \nATOM    293  CG1 VAL A  37      31.405   6.824  -4.758  1.00 60.38           C  \nATOM    294  CG2 VAL A  37      30.246   7.863  -6.765  1.00 74.03           C  \nATOM    295  N   VAL A  38      34.610   7.892  -6.066  1.00 64.06           N  \nATOM    296  CA  VAL A  38      35.867   7.378  -5.546  1.00 58.83           C  \nATOM    297  C   VAL A  38      36.857   8.549  -5.280  1.00 43.73           C  \nATOM    298  O   VAL A  38      37.622   8.506  -4.319  1.00 46.63           O  \nATOM    299  CB  VAL A  38      36.373   6.251  -6.467  1.00 65.19           C  \nATOM    300  CG1 VAL A  38      37.801   6.525  -6.920  1.00 49.93           C  \nATOM    301  CG2 VAL A  38      36.148   4.876  -5.782  1.00 56.92           C  \nATOM    302  N   THR A  39      36.758   9.639  -6.066  1.00 46.84           N  \nATOM    303  CA  THR A  39      37.567  10.847  -5.744  1.00 51.95           C  \nATOM    304  C   THR A  39      37.237  11.182  -4.261  1.00 43.72           C  \nATOM    305  O   THR A  39      38.088  11.053  -3.338  1.00 47.00           O  \nATOM    306  CB  THR A  39      37.280  12.114  -6.660  1.00 48.05           C  \nATOM    307  OG1 THR A  39      36.981  11.783  -8.056  1.00 45.51           O  \nATOM    308  CG2 THR A  39      38.433  13.129  -6.554  1.00 44.76           C  \nATOM    309  N   TYR A  40A     35.960  11.519  -4.047  1.00 42.75           N  \nATOM    310  CA  TYR A  40A     35.468  11.887  -2.719  1.00 42.39           C  \nATOM    311  C   TYR A  40A     35.865  10.960  -1.564  1.00 38.59           C  \nATOM    312  O   TYR A  40A     36.406  11.403  -0.487  1.00 39.56           O  \nATOM    313  CB  TYR A  40A     33.973  12.124  -2.834  1.00 46.65           C  \nATOM    314  CG  TYR A  40A     33.686  13.453  -3.505  1.00 43.41           C  \nATOM    315  CD1 TYR A  40A     33.879  14.637  -2.802  1.00 44.86           C  \nATOM    316  CD2 TYR A  40A     33.240  13.537  -4.841  1.00 41.98           C  \nATOM    317  CE1 TYR A  40A     33.624  15.866  -3.394  1.00 42.26           C  \nATOM    318  CE2 TYR A  40A     32.980  14.776  -5.447  1.00 40.42           C  \nATOM    319  CZ  TYR A  40A     33.203  15.935  -4.718  1.00 45.69           C  \nATOM    320  OH  TYR A  40A     32.984  17.193  -5.252  1.00 51.00           O  \nATOM    321  N   ASN A  41      35.710   9.651  -1.796  1.00 47.43           N  \nATOM    322  CA  ASN A  41      35.867   8.663  -0.706  1.00 42.53           C  \nATOM    323  C   ASN A  41      37.267   8.618  -0.268  1.00 42.91           C  \nATOM    324  O   ASN A  41      37.598   8.499   0.930  1.00 46.34           O  \nATOM    325  CB  ASN A  41      35.411   7.283  -1.226  1.00 59.70           C  \nATOM    326  CG  ASN A  41      33.892   7.070  -1.090  1.00 69.22           C  \nATOM    327  OD1 ASN A  41      33.395   6.907   0.008  1.00 65.66           O  \nATOM    328  ND2 ASN A  41      33.167   7.070  -2.204  1.00 78.98           N  \nATOM    329  N   THR A  42      38.159   8.771  -1.241  1.00 53.81           N  \nATOM    330  CA  THR A  42      39.597   8.662  -0.915  1.00 49.00           C  \nATOM    331  C   THR A  42      40.013   9.877  -0.127  1.00 36.26           C  \nATOM    332  O   THR A  42      40.714   9.777   0.862  1.00 41.31           O  \nATOM    333  CB  THR A  42      40.436   8.747  -2.187  1.00 59.90           C  \nATOM    334  OG1 THR A  42      39.878   7.907  -3.202  1.00 62.80           O  \nATOM    335  CG2 THR A  42      41.860   8.414  -1.907  1.00 56.62           C  \nATOM    336  N   LEU A  43      39.566  11.049  -0.569  1.00 44.66           N  \nATOM    337  CA  LEU A  43      39.869  12.303   0.198  1.00 38.38           C  \nATOM    338  C   LEU A  43      39.314  12.271   1.591  1.00 37.33           C  \nATOM    339  O   LEU A  43      40.032  12.520   2.600  1.00 41.19           O  \nATOM    340  CB  LEU A  43      39.355  13.491  -0.593  1.00 39.36           C  \nATOM    341  CG  LEU A  43      40.239  13.729  -1.858  1.00 45.11           C  \nATOM    342  CD1 LEU A  43      39.604  14.781  -2.785  1.00 33.82           C  \nATOM    343  CD2 LEU A  43      41.672  14.142  -1.441  1.00 41.74           C  \nATOM    344  N   ILE A  44      38.046  11.829   1.678  1.00 43.27           N  \nATOM    345  CA  ILE A  44      37.345  11.856   2.970  1.00 40.78           C  \nATOM    346  C   ILE A  44      38.032  10.916   3.899  1.00 38.28           C  \nATOM    347  O   ILE A  44      38.344  11.244   5.077  1.00 40.74           O  \nATOM    348  CB  ILE A  44      35.846  11.491   2.810  1.00 42.54           C  \nATOM    349  CG1 ILE A  44      35.113  12.701   2.211  1.00 39.81           C  \nATOM    350  CG2 ILE A  44      35.333  11.042   4.173  1.00 38.40           C  \nATOM    351  CD1 ILE A  44      33.798  12.425   1.449  1.00 42.65           C  \nATOM    352  N   ASP A  45      38.338   9.729   3.390  1.00 49.26           N  \nATOM    353  CA  ASP A  45      39.021   8.794   4.260  1.00 45.60           C  \nATOM    354  C   ASP A  45      40.380   9.287   4.679  1.00 50.46           C  \nATOM    355  O   ASP A  45      40.757   9.161   5.857  1.00 54.19           O  \nATOM    356  CB  ASP A  45      39.206   7.461   3.573  1.00 63.14           C  \nATOM    357  CG  ASP A  45      40.151   6.571   4.367  1.00 78.39           C  \nATOM    358  OD1 ASP A  45      39.945   6.413   5.604  1.00 70.17           O  \nATOM    359  OD2 ASP A  45      41.136   6.084   3.778  1.00 94.77           O  \nATOM    360  N   GLY A  46      41.126   9.909   3.752  1.00 50.41           N  \nATOM    361  CA  GLY A  46      42.426  10.472   4.187  1.00 38.66           C  \nATOM    362  C   GLY A  46      42.284  11.567   5.204  1.00 42.28           C  \nATOM    363  O   GLY A  46      42.990  11.586   6.206  1.00 44.66           O  \nATOM    364  N   LEU A  47      41.330  12.487   4.995  1.00 43.22           N  \nATOM    365  CA  LEU A  47      41.156  13.563   5.989  1.00 38.85           C  \nATOM    366  C   LEU A  47      40.756  13.034   7.381  1.00 44.61           C  \nATOM    367  O   LEU A  47      41.212  13.546   8.449  1.00 47.39           O  \nATOM    368  CB  LEU A  47      40.103  14.613   5.488  1.00 38.31           C  \nATOM    369  CG  LEU A  47      40.498  15.382   4.225  1.00 42.91           C  \nATOM    370  CD1 LEU A  47      39.300  16.056   3.520  1.00 35.49           C  \nATOM    371  CD2 LEU A  47      41.549  16.398   4.704  1.00 40.35           C  \nATOM    372  N   ALA A  48      39.821  12.077   7.371  1.00 51.27           N  \nATOM    373  CA  ALA A  48      39.325  11.493   8.615  1.00 51.00           C  \nATOM    374  C   ALA A  48      40.508  10.815   9.280  1.00 48.88           C  \nATOM    375  O   ALA A  48      40.814  11.164  10.396  1.00 51.69           O  \nATOM    376  CB  ALA A  48      38.197  10.500   8.351  1.00 48.97           C  \nATOM    377  N   LYS A  49      41.263   9.949   8.584  1.00 60.64           N  \nATOM    378  CA  LYS A  49      42.462   9.349   9.283  1.00 62.68           C  \nATOM    379  C   LYS A  49      43.403  10.396   9.858  1.00 65.89           C  \nATOM    380  O   LYS A  49      43.986  10.157  10.907  1.00 70.24           O  \nATOM    381  CB  LYS A  49      43.299   8.371   8.458  1.00 58.11           C  \nATOM    382  CG  LYS A  49      42.561   7.468   7.482  1.00 72.74           C  \nATOM    383  CD  LYS A  49      42.007   6.183   8.085  1.00 76.60           C  \nATOM    384  CE  LYS A  49      40.596   6.377   8.632  1.00 86.35           C  \nATOM    385  NZ  LYS A  49      40.272   5.297   9.596  1.00 80.39           N  \nATOM    386  N   ALA A  50      43.540  11.556   9.209  1.00 60.69           N  \nATOM    387  CA  ALA A  50      44.531  12.544   9.660  1.00 52.39           C  \nATOM    388  C   ALA A  50      43.996  13.533  10.695  1.00 58.42           C  \nATOM    389  O   ALA A  50      44.578  14.610  10.863  1.00 70.09           O  \nATOM    390  CB  ALA A  50      45.104  13.285   8.465  1.00 57.58           C  \nATOM    391  N   GLY A  51      42.886  13.201  11.365  1.00 52.26           N  \nATOM    392  CA  GLY A  51      42.240  14.124  12.322  1.00 52.53           C  \nATOM    393  C   GLY A  51      41.433  15.285  11.767  1.00 62.56           C  \nATOM    394  O   GLY A  51      40.883  16.090  12.529  1.00 70.47           O  \nATOM    395  N   ARG A  52      41.317  15.360  10.437  1.00 54.40           N  \nATOM    396  CA  ARG A  52      40.774  16.554   9.794  1.00 50.18           C  \nATOM    397  C   ARG A  52      39.318  16.336   9.432  1.00 50.03           C  \nATOM    398  O   ARG A  52      38.871  16.287   8.269  1.00 46.69           O  \nATOM    399  CB  ARG A  52      41.653  16.971   8.634  1.00 45.56           C  \nATOM    400  CG  ARG A  52      43.092  17.304   9.098  1.00 40.46           C  \nATOM    401  CD  ARG A  52      43.991  17.355   7.889  1.00 40.57           C  \nATOM    402  NE  ARG A  52      43.547  18.448   6.988  1.00 45.51           N  \nATOM    403  CZ  ARG A  52      44.263  18.836   5.938  1.00 46.87           C  \nATOM    404  NH1 ARG A  52      43.852  19.798   5.135  1.00 40.43           N  \nATOM    405  NH2 ARG A  52      45.414  18.230   5.683  1.00 56.56           N  \nATOM    406  N   LEU A  53      38.576  16.182  10.497  1.00 50.09           N  \nATOM    407  CA  LEU A  53      37.295  15.593  10.371  1.00 49.60           C  \nATOM    408  C   LEU A  53      36.332  16.678   9.897  1.00 46.14           C  \nATOM    409  O   LEU A  53      35.467  16.369   9.133  1.00 56.56           O  \nATOM    410  CB  LEU A  53      36.922  14.867  11.689  1.00 47.31           C  \nATOM    411  CG  LEU A  53      35.612  14.125  11.841  1.00 48.54           C  \nATOM    412  CD1 LEU A  53      35.569  12.970  10.833  1.00 44.24           C  \nATOM    413  CD2 LEU A  53      35.335  13.679  13.279  1.00 41.87           C  \nATOM    414  N   GLU A  54      36.534  17.948  10.263  1.00 42.40           N  \nATOM    415  CA  GLU A  54      35.705  19.018   9.752  1.00 41.78           C  \nATOM    416  C   GLU A  54      35.819  19.132   8.240  1.00 46.72           C  \nATOM    417  O   GLU A  54      34.827  19.306   7.511  1.00 41.38           O  \nATOM    418  CB  GLU A  54      36.090  20.360  10.317  1.00 41.17           C  \nATOM    419  CG  GLU A  54      35.621  20.590  11.725  1.00 59.58           C  \nATOM    420  CD  GLU A  54      34.128  20.389  11.838  1.00 55.44           C  \nATOM    421  OE1 GLU A  54      33.355  21.180  11.291  1.00 57.08           O  \nATOM    422  OE2 GLU A  54      33.728  19.403  12.441  1.00 67.03           O  \nATOM    423  N   GLU A  55      37.028  19.055   7.727  1.00 37.17           N  \nATOM    424  CA  GLU A  55      37.116  19.070   6.253  1.00 34.75           C  \nATOM    425  C   GLU A  55      36.554  17.802   5.602  1.00 33.62           C  \nATOM    426  O   GLU A  55      36.054  17.851   4.479  1.00 33.55           O  \nATOM    427  CB  GLU A  55      38.557  19.304   5.817  1.00 38.77           C  \nATOM    428  CG  GLU A  55      39.146  20.606   6.369  1.00 41.22           C  \nATOM    429  CD  GLU A  55      40.656  20.559   6.305  1.00 51.43           C  \nATOM    430  OE1 GLU A  55      41.322  20.235   7.321  1.00 52.21           O  \nATOM    431  OE2 GLU A  55      41.159  20.732   5.166  1.00 49.47           O  \nATOM    432  N   ALA A  56      36.636  16.659   6.280  1.00 33.32           N  \nATOM    433  CA  ALA A  56      36.069  15.438   5.699  1.00 35.70           C  \nATOM    434  C   ALA A  56      34.549  15.631   5.561  1.00 32.24           C  \nATOM    435  O   ALA A  56      33.960  15.321   4.521  1.00 35.36           O  \nATOM    436  CB  ALA A  56      36.380  14.196   6.561  1.00 30.64           C  \nATOM    437  N   LEU A  57      33.937  16.162   6.596  1.00 35.60           N  \nATOM    438  CA  LEU A  57      32.494  16.471   6.592  1.00 40.08           C  \nATOM    439  C   LEU A  57      32.080  17.477   5.544  1.00 39.54           C  \nATOM    440  O   LEU A  57      31.059  17.259   4.889  1.00 40.95           O  \nATOM    441  CB  LEU A  57      32.003  16.922   7.982  1.00 40.01           C  \nATOM    442  CG  LEU A  57      32.025  15.638   8.812  1.00 43.72           C  \nATOM    443  CD1 LEU A  57      32.307  15.936  10.261  1.00 44.96           C  \nATOM    444  CD2 LEU A  57      30.687  14.979   8.676  1.00 43.79           C  \nATOM    445  N   GLN A  58      32.891  18.503   5.292  1.00 38.51           N  \nATOM    446  CA  GLN A  58      32.561  19.446   4.216  1.00 41.21           C  \nATOM    447  C   GLN A  58      32.668  18.774   2.871  1.00 40.69           C  \nATOM    448  O   GLN A  58      31.897  19.036   1.988  1.00 42.84           O  \nATOM    449  CB  GLN A  58      33.562  20.627   4.185  1.00 40.70           C  \nATOM    450  CG  GLN A  58      33.632  21.218   5.558  1.00 62.18           C  \nATOM    451  CD  GLN A  58      34.618  22.353   5.664  1.00 71.83           C  \nATOM    452  OE1 GLN A  58      35.527  22.503   4.821  1.00 66.37           O  \nATOM    453  NE2 GLN A  58      34.446  23.165   6.711  1.00 56.63           N  \nATOM    454  N   LEU A  59      33.648  17.904   2.688  1.00 34.72           N  \nATOM    455  CA  LEU A  59      33.732  17.291   1.355  1.00 35.75           C  \nATOM    456  C   LEU A  59      32.536  16.400   1.099  1.00 34.60           C  \nATOM    457  O   LEU A  59      32.064  16.321   0.002  1.00 30.56           O  \nATOM    458  CB  LEU A  59      35.016  16.461   1.189  1.00 34.10           C  \nATOM    459  CG  LEU A  59      36.267  17.110   0.667  1.00 47.49           C  \nATOM    460  CD1 LEU A  59      37.218  15.977   0.384  1.00 52.11           C  \nATOM    461  CD2 LEU A  59      35.917  17.855  -0.615  1.00 53.37           C  \nATOM    462  N   PHE A  60      32.138  15.659   2.128  1.00 33.98           N  \nATOM    463  CA  PHE A  60      30.940  14.826   2.099  1.00 32.72           C  \nATOM    464  C   PHE A  60      29.670  15.591   1.697  1.00 35.42           C  \nATOM    465  O   PHE A  60      28.925  15.158   0.848  1.00 34.06           O  \nATOM    466  CB  PHE A  60      30.797  14.247   3.506  1.00 34.29           C  \nATOM    467  CG  PHE A  60      29.492  13.576   3.777  1.00 39.45           C  \nATOM    468  CD1 PHE A  60      29.190  12.367   3.196  1.00 40.13           C  \nATOM    469  CD2 PHE A  60      28.594  14.135   4.666  1.00 47.44           C  \nATOM    470  CE1 PHE A  60      27.990  11.707   3.481  1.00 44.04           C  \nATOM    471  CE2 PHE A  60      27.364  13.515   4.934  1.00 47.62           C  \nATOM    472  CZ  PHE A  60      27.085  12.288   4.357  1.00 47.93           C  \nATOM    473  N   GLN A  61      29.439  16.753   2.297  1.00 38.99           N  \nATOM    474  CA  GLN A  61      28.365  17.627   1.850  1.00 38.41           C  \nATOM    475  C   GLN A  61      28.558  18.016   0.398  1.00 44.31           C  \nATOM    476  O   GLN A  61      27.612  18.003  -0.425  1.00 43.26           O  \nATOM    477  CB  GLN A  61      28.346  18.893   2.750  1.00 41.47           C  \nATOM    478  CG  GLN A  61      28.002  18.577   4.225  1.00 40.49           C  \nATOM    479  CD  GLN A  61      26.648  17.775   4.409  1.00 51.16           C  \nATOM    480  OE1 GLN A  61      26.035  17.332   3.446  1.00 56.30           O  \nATOM    481  NE2 GLN A  61      26.199  17.590   5.668  1.00 47.93           N  \nATOM    482  N   GLU A  62      29.783  18.381   0.033  1.00 50.33           N  \nATOM    483  CA  GLU A  62      30.001  18.857  -1.350  1.00 44.06           C  \nATOM    484  C   GLU A  62      29.692  17.711  -2.327  1.00 44.12           C  \nATOM    485  O   GLU A  62      29.001  17.854  -3.334  1.00 48.51           O  \nATOM    486  CB  GLU A  62      31.400  19.370  -1.483  1.00 41.37           C  \nATOM    487  CG  GLU A  62      31.728  19.790  -2.885  1.00 50.19           C  \nATOM    488  CD  GLU A  62      33.235  19.955  -3.073  1.00 53.89           C  \nATOM    489  OE1 GLU A  62      33.906  20.532  -2.180  1.00 61.68           O  \nATOM    490  OE2 GLU A  62      33.763  19.467  -4.088  1.00 55.39           O  \nATOM    491  N   MET A  63      30.132  16.538  -1.954  1.00 43.34           N  \nATOM    492  CA  MET A  63      29.880  15.353  -2.752  1.00 47.77           C  \nATOM    493  C   MET A  63      28.345  15.224  -3.017  1.00 54.67           C  \nATOM    494  O   MET A  63      27.919  14.947  -4.135  1.00 46.79           O  \nATOM    495  CB  MET A  63      30.322  14.213  -1.890  1.00 41.78           C  \nATOM    496  CG  MET A  63      30.260  12.885  -2.502  1.00 55.70           C  \nATOM    497  SD  MET A  63      30.761  11.837  -1.108  1.00 58.37           S  \nATOM    498  CE  MET A  63      30.667  10.400  -2.135  1.00 43.56           C  \nATOM    499  N   LYS A  64      27.522  15.421  -1.978  1.00 52.98           N  \nATOM    500  CA  LYS A  64      26.067  15.210  -2.105  1.00 50.67           C  \nATOM    501  C   LYS A  64      25.410  16.269  -2.988  1.00 54.36           C  \nATOM    502  O   LYS A  64      24.472  15.957  -3.743  1.00 51.01           O  \nATOM    503  CB  LYS A  64      25.393  15.109  -0.728  1.00 42.52           C  \nATOM    504  CG  LYS A  64      25.548  13.679  -0.177  1.00 44.95           C  \nATOM    505  CD  LYS A  64      24.912  13.532   1.208  1.00 47.35           C  \nATOM    506  CE  LYS A  64      25.441  14.576   2.174  1.00 50.76           C  \nATOM    507  NZ  LYS A  64      24.668  14.437   3.439  1.00 51.85           N  \nATOM    508  N   GLU A  65      25.939  17.491  -2.935  1.00 42.68           N  \nATOM    509  CA  GLU A  65      25.422  18.577  -3.732  1.00 49.97           C  \nATOM    510  C   GLU A  65      25.734  18.341  -5.196  1.00 59.57           C  \nATOM    511  O   GLU A  65      24.903  18.644  -6.027  1.00 65.19           O  \nATOM    512  CB  GLU A  65      25.988  19.933  -3.340  1.00 41.13           C  \nATOM    513  CG  GLU A  65      25.796  20.321  -1.890  1.00 42.79           C  \nATOM    514  CD  GLU A  65      24.365  20.710  -1.564  1.00 49.12           C  \nATOM    515  OE1 GLU A  65      23.539  20.705  -2.487  1.00 46.18           O  \nATOM    516  OE2 GLU A  65      24.063  21.061  -0.397  1.00 50.98           O  \nATOM    517  N   LYS A  66      26.909  17.796  -5.515  1.00 54.95           N  \nATOM    518  CA  LYS A  66      27.289  17.586  -6.922  1.00 59.00           C  \nATOM    519  C   LYS A  66      26.651  16.332  -7.530  1.00 58.06           C  \nATOM    520  O   LYS A  66      26.939  15.955  -8.651  1.00 65.74           O  \nATOM    521  CB  LYS A  66      28.829  17.610  -7.104  1.00 61.51           C  \nATOM    522  CG  LYS A  66      29.427  18.965  -6.755  1.00 61.52           C  \nATOM    523  CD  LYS A  66      30.900  19.007  -7.103  1.00 83.14           C  \nATOM    524  CE  LYS A  66      31.440  20.434  -7.120  1.00 82.54           C  \nATOM    525  NZ  LYS A  66      32.643  20.446  -7.995  1.00 86.31           N  \nATOM    526  N   GLY A  67      25.772  15.687  -6.791  1.00 57.11           N  \nATOM    527  CA  GLY A  67      25.036  14.568  -7.326  1.00 59.30           C  \nATOM    528  C   GLY A  67      25.904  13.329  -7.344  1.00 66.21           C  \nATOM    529  O   GLY A  67      25.487  12.277  -7.828  1.00 66.85           O  \nATOM    530  N   VAL A  68      27.111  13.433  -6.793  1.00 67.82           N  \nATOM    531  CA  VAL A  68      27.907  12.238  -6.520  1.00 60.26           C  \nATOM    532  C   VAL A  68      27.416  11.473  -5.244  1.00 69.68           C  \nATOM    533  O   VAL A  68      27.402  12.028  -4.151  1.00 79.83           O  \nATOM    534  CB  VAL A  68      29.426  12.565  -6.495  1.00 59.49           C  \nATOM    535  CG1 VAL A  68      30.209  11.268  -6.282  1.00 48.34           C  \nATOM    536  CG2 VAL A  68      29.880  13.322  -7.782  1.00 50.81           C  \nATOM    537  N   LYS A  69      27.053  10.195  -5.393  1.00 78.45           N  \nATOM    538  CA  LYS A  69      26.427   9.350  -4.333  1.00 78.42           C  \nATOM    539  C   LYS A  69      27.349   8.716  -3.220  1.00 75.20           C  \nATOM    540  O   LYS A  69      28.164   7.809  -3.501  1.00 79.02           O  \nATOM    541  CB  LYS A  69      25.559   8.220  -4.997  1.00 80.99           C  \nATOM    542  CG  LYS A  69      26.335   7.272  -5.952  1.00 96.85           C  \nATOM    543  CD  LYS A  69      25.552   6.022  -6.403  1.00 94.06           C  \nATOM    544  CE  LYS A  69      24.693   6.305  -7.637  1.00101.77           C  \nATOM    545  NZ  LYS A  69      23.804   7.484  -7.385  1.00113.17           N  \nATOM    546  N   PRO A  70      27.184   9.142  -1.939  1.00 69.32           N  \nATOM    547  CA  PRO A  70      27.830   8.412  -0.814  1.00 66.74           C  \nATOM    548  C   PRO A  70      27.465   6.914  -0.710  1.00 64.00           C  \nATOM    549  O   PRO A  70      26.348   6.553  -1.045  1.00 64.27           O  \nATOM    550  CB  PRO A  70      27.315   9.173   0.429  1.00 64.21           C  \nATOM    551  CG  PRO A  70      27.078  10.555  -0.064  1.00 62.62           C  \nATOM    552  CD  PRO A  70      26.585  10.420  -1.485  1.00 63.32           C  \nATOM    553  N   ASP A  71      28.378   6.078  -0.202  1.00 57.27           N  \nATOM    554  CA  ASP A  71      28.144   4.649  -0.019  1.00 54.72           C  \nATOM    555  C   ASP A  71      28.639   4.188   1.386  1.00 63.71           C  \nATOM    556  O   ASP A  71      29.041   5.010   2.216  1.00 58.69           O  \nATOM    557  CB  ASP A  71      28.804   3.849  -1.149  1.00 67.79           C  \nATOM    558  CG  ASP A  71      30.327   3.723  -0.975  1.00 74.76           C  \nATOM    559  OD1 ASP A  71      30.963   4.710  -0.583  1.00 65.85           O  \nATOM    560  OD2 ASP A  71      30.893   2.637  -1.222  1.00 87.85           O  \nATOM    561  N   VAL A  72      28.589   2.890   1.675  1.00 58.99           N  \nATOM    562  CA  VAL A  72      28.746   2.433   3.048  1.00 60.28           C  \nATOM    563  C   VAL A  72      30.142   2.748   3.580  1.00 61.43           C  \nATOM    564  O   VAL A  72      30.335   2.921   4.797  1.00 59.55           O  \nATOM    565  CB  VAL A  72      28.453   0.927   3.188  1.00 55.74           C  \nATOM    566  CG1 VAL A  72      28.170   0.560   4.662  1.00 51.54           C  \nATOM    567  CG2 VAL A  72      27.282   0.572   2.301  1.00 71.62           C  \nATOM    568  N   VAL A  73      31.117   2.824   2.683  1.00 55.26           N  \nATOM    569  CA  VAL A  73      32.422   3.239   3.160  1.00 56.51           C  \nATOM    570  C   VAL A  73      32.478   4.746   3.486  1.00 43.77           C  \nATOM    571  O   VAL A  73      33.037   5.087   4.486  1.00 50.87           O  \nATOM    572  CB  VAL A  73      33.554   2.854   2.215  1.00 58.58           C  \nATOM    573  CG1 VAL A  73      33.882   1.391   2.454  1.00 62.34           C  \nATOM    574  CG2 VAL A  73      33.162   3.173   0.778  1.00 59.07           C  \nATOM    575  N   THR A  74      31.909   5.610   2.653  1.00 39.47           N  \nATOM    576  CA  THR A  74      31.809   7.024   2.984  1.00 52.82           C  \nATOM    577  C   THR A  74      31.261   7.142   4.431  1.00 44.49           C  \nATOM    578  O   THR A  74      31.947   7.581   5.339  1.00 46.69           O  \nATOM    579  CB  THR A  74      30.863   7.824   2.061  1.00 51.58           C  \nATOM    580  OG1 THR A  74      30.843   7.297   0.724  1.00 54.42           O  \nATOM    581  CG2 THR A  74      31.292   9.345   2.031  1.00 51.39           C  \nATOM    582  N   TYR A  75      30.051   6.618   4.618  1.00 43.15           N  \nATOM    583  CA  TYR A  75      29.333   6.693   5.855  1.00 37.18           C  \nATOM    584  C   TYR A  75      30.087   6.112   7.002  1.00 37.42           C  \nATOM    585  O   TYR A  75      30.180   6.758   8.100  1.00 37.47           O  \nATOM    586  CB  TYR A  75      27.921   6.101   5.676  1.00 35.24           C  \nATOM    587  CG  TYR A  75      26.953   7.028   4.946  1.00 41.87           C  \nATOM    588  CD1 TYR A  75      26.420   8.120   5.612  1.00 36.19           C  \nATOM    589  CD2 TYR A  75      26.482   6.753   3.654  1.00 39.67           C  \nATOM    590  CE1 TYR A  75      25.548   8.968   5.012  1.00 36.91           C  \nATOM    591  CE2 TYR A  75      25.551   7.613   3.044  1.00 41.66           C  \nATOM    592  CZ  TYR A  75      25.102   8.721   3.737  1.00 42.71           C  \nATOM    593  OH  TYR A  75      24.209   9.659   3.211  1.00 45.71           O  \nATOM    594  N   ASN A  76      30.695   4.949   6.789  1.00 40.04           N  \nATOM    595  CA  ASN A  76      31.423   4.275   7.915  1.00 36.85           C  \nATOM    596  C   ASN A  76      32.628   5.061   8.398  1.00 39.40           C  \nATOM    597  O   ASN A  76      32.947   5.124   9.617  1.00 40.15           O  \nATOM    598  CB  ASN A  76      31.881   2.841   7.517  1.00 44.48           C  \nATOM    599  CG  ASN A  76      30.748   1.738   7.683  1.00 57.38           C  \nATOM    600  OD1 ASN A  76      29.585   2.021   7.997  1.00 42.11           O  \nATOM    601  ND2 ASN A  76      31.128   0.471   7.476  1.00 56.94           N  \nATOM    602  N   THR A  77      33.346   5.635   7.443  1.00 43.30           N  \nATOM    603  CA  THR A  77      34.554   6.429   7.812  1.00 47.06           C  \nATOM    604  C   THR A  77      34.144   7.642   8.619  1.00 38.79           C  \nATOM    605  O   THR A  77      34.759   7.950   9.618  1.00 39.87           O  \nATOM    606  CB  THR A  77      35.237   7.041   6.570  1.00 52.15           C  \nATOM    607  OG1 THR A  77      35.335   6.036   5.586  1.00 51.59           O  \nATOM    608  CG2 THR A  77      36.662   7.528   6.900  1.00 53.28           C  \nATOM    609  N   LEU A  78      33.056   8.293   8.181  1.00 36.61           N  \nATOM    610  CA  LEU A  78      32.636   9.511   8.841  1.00 35.69           C  \nATOM    611  C   LEU A  78      32.100   9.158  10.155  1.00 33.13           C  \nATOM    612  O   LEU A  78      32.405   9.797  11.178  1.00 31.93           O  \nATOM    613  CB  LEU A  78      31.595  10.261   7.995  1.00 33.18           C  \nATOM    614  CG  LEU A  78      32.274  10.846   6.788  1.00 39.02           C  \nATOM    615  CD1 LEU A  78      31.204  11.348   5.820  1.00 32.37           C  \nATOM    616  CD2 LEU A  78      33.285  11.984   7.143  1.00 36.60           C  \nATOM    617  N   ILE A  79      31.245   8.141  10.165  1.00 36.80           N  \nATOM    618  CA  ILE A  79      30.638   7.791  11.433  1.00 37.80           C  \nATOM    619  C   ILE A  79      31.743   7.368  12.379  1.00 36.97           C  \nATOM    620  O   ILE A  79      31.747   7.694  13.602  1.00 37.42           O  \nATOM    621  CB  ILE A  79      29.607   6.655  11.298  1.00 34.80           C  \nATOM    622  CG1 ILE A  79      28.353   7.165  10.551  1.00 37.99           C  \nATOM    623  CG2 ILE A  79      29.353   6.120  12.701  1.00 35.49           C  \nATOM    624  CD1 ILE A  79      27.459   6.053   9.959  1.00 39.86           C  \nATOM    625  N   ASP A  80      32.733   6.687  11.829  1.00 39.56           N  \nATOM    626  CA  ASP A  80      33.790   6.275  12.764  1.00 49.22           C  \nATOM    627  C   ASP A  80      34.638   7.466  13.252  1.00 43.54           C  \nATOM    628  O   ASP A  80      34.963   7.529  14.432  1.00 40.58           O  \nATOM    629  CB  ASP A  80      34.686   5.199  12.164  1.00 57.67           C  \nATOM    630  CG  ASP A  80      35.865   4.856  13.087  1.00 74.67           C  \nATOM    631  OD1 ASP A  80      35.636   4.201  14.142  1.00 62.18           O  \nATOM    632  OD2 ASP A  80      37.015   5.286  12.775  1.00 73.53           O  \nATOM    633  N   GLY A  81      34.990   8.432  12.376  1.00 43.78           N  \nATOM    634  CA  GLY A  81      35.761   9.612  12.895  1.00 35.18           C  \nATOM    635  C   GLY A  81      34.888  10.311  13.925  1.00 36.83           C  \nATOM    636  O   GLY A  81      35.335  10.757  14.973  1.00 43.33           O  \nATOM    637  N   LEU A  82      33.589  10.444  13.648  1.00 37.47           N  \nATOM    638  CA  LEU A  82      32.770  11.224  14.608  1.00 38.22           C  \nATOM    639  C   LEU A  82      32.705  10.528  15.947  1.00 42.54           C  \nATOM    640  O   LEU A  82      32.756  11.196  17.051  1.00 37.91           O  \nATOM    641  CB  LEU A  82      31.299  11.477  14.055  1.00 38.25           C  \nATOM    642  CG  LEU A  82      31.302  12.483  12.869  1.00 39.10           C  \nATOM    643  CD1 LEU A  82      30.189  12.183  11.889  1.00 39.43           C  \nATOM    644  CD2 LEU A  82      31.179  13.895  13.465  1.00 40.84           C  \nATOM    645  N   ALA A  83      32.498   9.197  15.871  1.00 40.48           N  \nATOM    646  CA  ALA A  83      32.328   8.414  17.136  1.00 50.27           C  \nATOM    647  C   ALA A  83      33.625   8.506  17.916  1.00 49.82           C  \nATOM    648  O   ALA A  83      33.610   8.840  19.088  1.00 56.02           O  \nATOM    649  CB  ALA A  83      31.904   6.947  16.897  1.00 43.56           C  \nATOM    650  N   LYS A  84      34.759   8.349  17.254  1.00 57.06           N  \nATOM    651  CA  LYS A  84      35.998   8.442  18.016  1.00 60.51           C  \nATOM    652  C   LYS A  84      36.257   9.822  18.581  1.00 64.66           C  \nATOM    653  O   LYS A  84      36.866   9.904  19.623  1.00 61.84           O  \nATOM    654  CB  LYS A  84      37.169   7.836  17.264  1.00 59.43           C  \nATOM    655  CG  LYS A  84      37.049   6.297  17.294  1.00 72.40           C  \nATOM    656  CD  LYS A  84      37.985   5.627  16.305  1.00 77.78           C  \nATOM    657  CE  LYS A  84      39.014   6.660  15.840  1.00 82.77           C  \nATOM    658  NZ  LYS A  84      40.049   6.113  14.922  1.00 84.93           N  \nATOM    659  N   ALA A  85      35.735  10.899  17.966  1.00 63.38           N  \nATOM    660  CA  ALA A  85      35.892  12.265  18.546  1.00 46.15           C  \nATOM    661  C   ALA A  85      34.870  12.631  19.600  1.00 50.64           C  \nATOM    662  O   ALA A  85      34.718  13.796  19.917  1.00 63.52           O  \nATOM    663  CB  ALA A  85      35.904  13.336  17.461  1.00 44.47           C  \nATOM    664  N   GLY A  86      34.144  11.668  20.152  1.00 49.45           N  \nATOM    665  CA  GLY A  86      33.028  12.004  21.051  1.00 46.17           C  \nATOM    666  C   GLY A  86      31.764  12.544  20.340  1.00 58.41           C  \nATOM    667  O   GLY A  86      30.712  12.730  20.982  1.00 50.06           O  \nATOM    668  N   ARG A  87      31.815  12.759  19.020  1.00 48.29           N  \nATOM    669  CA  ARG A  87      30.704  13.431  18.351  1.00 46.24           C  \nATOM    670  C   ARG A  87      29.580  12.485  17.961  1.00 39.29           C  \nATOM    671  O   ARG A  87      29.230  12.279  16.792  1.00 38.99           O  \nATOM    672  CB  ARG A  87      31.204  14.259  17.195  1.00 37.02           C  \nATOM    673  CG  ARG A  87      32.101  15.398  17.701  1.00 42.29           C  \nATOM    674  CD  ARG A  87      32.863  16.020  16.526  1.00 42.42           C  \nATOM    675  NE  ARG A  87      31.939  16.636  15.559  1.00 40.75           N  \nATOM    676  CZ  ARG A  87      32.405  17.335  14.516  1.00 46.24           C  \nATOM    677  NH1 ARG A  87      33.708  17.465  14.407  1.00 39.61           N  \nATOM    678  NH2 ARG A  87      31.603  17.928  13.604  1.00 38.26           N  \nATOM    679  N   LEU A  88      28.947  11.990  18.987  1.00 40.94           N  \nATOM    680  CA  LEU A  88      28.022  10.904  18.897  1.00 42.81           C  \nATOM    681  C   LEU A  88      26.701  11.308  18.313  1.00 42.19           C  \nATOM    682  O   LEU A  88      26.043  10.485  17.633  1.00 48.63           O  \nATOM    683  CB  LEU A  88      27.841  10.372  20.329  1.00 50.02           C  \nATOM    684  CG  LEU A  88      27.529   8.907  20.537  1.00 57.92           C  \nATOM    685  CD1 LEU A  88      28.403   8.038  19.630  1.00 51.08           C  \nATOM    686  CD2 LEU A  88      27.725   8.593  22.024  1.00 59.80           C  \nATOM    687  N   GLU A  89      26.251  12.544  18.576  1.00 38.39           N  \nATOM    688  CA  GLU A  89      24.912  12.918  18.029  1.00 39.33           C  \nATOM    689  C   GLU A  89      25.017  13.026  16.521  1.00 33.88           C  \nATOM    690  O   GLU A  89      24.131  12.649  15.764  1.00 37.37           O  \nATOM    691  CB  GLU A  89      24.419  14.253  18.587  1.00 39.79           C  \nATOM    692  CG  GLU A  89      24.150  14.228  20.053  1.00 49.77           C  \nATOM    693  CD  GLU A  89      23.204  13.104  20.410  1.00 54.16           C  \nATOM    694  OE1 GLU A  89      23.679  12.169  21.060  1.00 66.76           O  \nATOM    695  OE2 GLU A  89      22.030  13.100  19.978  1.00 53.95           O  \nATOM    696  N   GLU A  90      26.115  13.586  16.066  1.00 30.39           N  \nATOM    697  CA  GLU A  90      26.251  13.701  14.646  1.00 33.40           C  \nATOM    698  C   GLU A  90      26.502  12.294  13.968  1.00 40.70           C  \nATOM    699  O   GLU A  90      26.016  12.001  12.836  1.00 32.83           O  \nATOM    700  CB  GLU A  90      27.444  14.633  14.428  1.00 34.91           C  \nATOM    701  CG  GLU A  90      27.453  15.028  13.001  1.00 44.20           C  \nATOM    702  CD  GLU A  90      28.304  16.258  12.692  1.00 56.52           C  \nATOM    703  OE1 GLU A  90      28.899  16.982  13.622  1.00 39.95           O  \nATOM    704  OE2 GLU A  90      28.362  16.438  11.433  1.00 47.18           O  \nATOM    705  N   ALA A  91      27.314  11.453  14.632  1.00 35.46           N  \nATOM    706  CA  ALA A  91      27.479  10.027  14.166  1.00 39.20           C  \nATOM    707  C   ALA A  91      26.130   9.316  14.053  1.00 30.08           C  \nATOM    708  O   ALA A  91      25.824   8.695  13.027  1.00 31.49           O  \nATOM    709  CB  ALA A  91      28.445   9.249  15.101  1.00 37.84           C  \nATOM    710  N   LEU A  92      25.260   9.519  15.034  1.00 34.58           N  \nATOM    711  CA  LEU A  92      23.902   8.891  14.983  1.00 35.24           C  \nATOM    712  C   LEU A  92      23.068   9.460  13.889  1.00 40.18           C  \nATOM    713  O   LEU A  92      22.257   8.766  13.224  1.00 34.56           O  \nATOM    714  CB  LEU A  92      23.179   9.100  16.308  1.00 38.48           C  \nATOM    715  CG  LEU A  92      23.850   8.185  17.376  1.00 40.03           C  \nATOM    716  CD1 LEU A  92      23.411   8.518  18.773  1.00 40.71           C  \nATOM    717  CD2 LEU A  92      23.345   6.793  17.055  1.00 43.32           C  \nATOM    718  N   GLN A  93      23.236  10.765  13.680  1.00 36.58           N  \nATOM    719  CA  GLN A  93      22.477  11.431  12.620  1.00 35.19           C  \nATOM    720  C   GLN A  93      22.909  10.907  11.269  1.00 33.23           C  \nATOM    721  O   GLN A  93      22.090  10.696  10.381  1.00 34.51           O  \nATOM    722  CB  GLN A  93      22.753  12.979  12.704  1.00 38.36           C  \nATOM    723  CG  GLN A  93      21.949  13.727  11.667  1.00 41.67           C  \nATOM    724  CD  GLN A  93      20.423  13.476  11.800  1.00 50.87           C  \nATOM    725  OE1 GLN A  93      19.844  13.655  12.866  1.00 48.12           O  \nATOM    726  NE2 GLN A  93      19.790  13.042  10.715  1.00 48.14           N  \nATOM    727  N   LEU A  94      24.213  10.731  11.051  1.00 30.78           N  \nATOM    728  CA  LEU A  94      24.612  10.166   9.740  1.00 31.91           C  \nATOM    729  C   LEU A  94      24.122   8.704   9.537  1.00 28.40           C  \nATOM    730  O   LEU A  94      23.884   8.277   8.412  1.00 36.43           O  \nATOM    731  CB  LEU A  94      26.178  10.166   9.586  1.00 30.76           C  \nATOM    732  CG  LEU A  94      26.869  11.465   9.297  1.00 45.34           C  \nATOM    733  CD1 LEU A  94      28.237  11.101   8.752  1.00 44.03           C  \nATOM    734  CD2 LEU A  94      26.060  12.273   8.262  1.00 41.85           C  \nATOM    735  N   PHE A  95      24.011   7.956  10.642  1.00 35.34           N  \nATOM    736  CA  PHE A  95      23.563   6.536  10.648  1.00 36.29           C  \nATOM    737  C   PHE A  95      22.109   6.509  10.202  1.00 36.23           C  \nATOM    738  O   PHE A  95      21.720   5.802   9.297  1.00 37.00           O  \nATOM    739  CB  PHE A  95      23.643   6.023  12.079  1.00 38.57           C  \nATOM    740  CG  PHE A  95      23.108   4.610  12.300  1.00 40.39           C  \nATOM    741  CD1 PHE A  95      23.694   3.508  11.677  1.00 42.18           C  \nATOM    742  CD2 PHE A  95      22.062   4.399  13.208  1.00 47.43           C  \nATOM    743  CE1 PHE A  95      23.211   2.206  11.931  1.00 43.49           C  \nATOM    744  CE2 PHE A  95      21.589   3.102  13.492  1.00 46.55           C  \nATOM    745  CZ  PHE A  95      22.167   2.013  12.835  1.00 35.96           C  \nATOM    746  N   GLN A  96      21.309   7.347  10.819  1.00 37.44           N  \nATOM    747  CA  GLN A  96      19.920   7.495  10.347  1.00 37.65           C  \nATOM    748  C   GLN A  96      19.841   7.939   8.874  1.00 42.78           C  \nATOM    749  O   GLN A  96      19.139   7.341   8.037  1.00 41.83           O  \nATOM    750  CB  GLN A  96      19.342   8.541  11.312  1.00 38.74           C  \nATOM    751  CG  GLN A  96      18.076   9.166  10.842  1.00 52.53           C  \nATOM    752  CD  GLN A  96      16.894   8.438  11.414  1.00 71.05           C  \nATOM    753  OE1 GLN A  96      16.705   7.219  11.162  1.00 61.63           O  \nATOM    754  NE2 GLN A  96      16.095   9.161  12.228  1.00 76.34           N  \nATOM    755  N   GLU A  97      20.635   8.949   8.495  1.00 38.94           N  \nATOM    756  CA  GLU A  97      20.662   9.352   7.054  1.00 36.80           C  \nATOM    757  C   GLU A  97      21.083   8.208   6.122  1.00 40.26           C  \nATOM    758  O   GLU A  97      20.515   8.040   5.034  1.00 38.54           O  \nATOM    759  CB  GLU A  97      21.642  10.546   6.853  1.00 34.69           C  \nATOM    760  CG  GLU A  97      21.731  11.028   5.406  1.00 40.90           C  \nATOM    761  CD  GLU A  97      22.824  12.086   5.236  1.00 48.02           C  \nATOM    762  OE1 GLU A  97      22.900  12.971   6.095  1.00 52.53           O  \nATOM    763  OE2 GLU A  97      23.655  11.975   4.304  1.00 51.85           O  \nATOM    764  N   MET A  98      22.135   7.468   6.491  1.00 37.89           N  \nATOM    765  CA  MET A  98      22.599   6.358   5.634  1.00 43.81           C  \nATOM    766  C   MET A  98      21.351   5.454   5.372  1.00 43.31           C  \nATOM    767  O   MET A  98      21.087   5.012   4.253  1.00 42.71           O  \nATOM    768  CB  MET A  98      23.638   5.534   6.434  1.00 45.10           C  \nATOM    769  CG  MET A  98      24.508   4.582   5.618  1.00 47.14           C  \nATOM    770  SD  MET A  98      25.609   3.615   6.735  1.00 54.67           S  \nATOM    771  CE  MET A  98      24.862   4.266   8.062  1.00 31.16           C  \nATOM    772  N   LYS A  99      20.579   5.207   6.430  1.00 45.42           N  \nATOM    773  CA  LYS A  99      19.402   4.293   6.319  1.00 54.14           C  \nATOM    774  C   LYS A  99      18.290   4.873   5.485  1.00 52.32           C  \nATOM    775  O   LYS A  99      17.764   4.190   4.619  1.00 50.51           O  \nATOM    776  CB  LYS A  99      18.811   3.953   7.663  1.00 47.14           C  \nATOM    777  CG  LYS A  99      19.721   3.119   8.502  1.00 50.41           C  \nATOM    778  CD  LYS A  99      18.858   2.658   9.637  1.00 48.41           C  \nATOM    779  CE  LYS A  99      19.584   2.758  10.930  1.00 50.39           C  \nATOM    780  NZ  LYS A  99      18.574   2.738  12.025  1.00 50.42           N  \nATOM    781  N   GLU A 100      17.954   6.141   5.706  1.00 52.93           N  \nATOM    782  CA  GLU A 100      17.040   6.791   4.787  1.00 51.58           C  \nATOM    783  C   GLU A 100      17.501   6.781   3.349  1.00 51.55           C  \nATOM    784  O   GLU A 100      16.665   6.644   2.490  1.00 58.02           O  \nATOM    785  CB  GLU A 100      16.725   8.184   5.232  1.00 52.82           C  \nATOM    786  CG  GLU A 100      16.081   8.117   6.579  1.00 55.79           C  \nATOM    787  CD  GLU A 100      15.800   9.478   7.150  1.00 70.44           C  \nATOM    788  OE1 GLU A 100      16.213  10.509   6.559  1.00 87.40           O  \nATOM    789  OE2 GLU A 100      15.169   9.512   8.224  1.00 73.15           O  \nATOM    790  N   LYS A 101      18.799   6.927   3.058  1.00 49.86           N  \nATOM    791  CA  LYS A 101      19.245   6.836   1.649  1.00 50.67           C  \nATOM    792  C   LYS A 101      19.198   5.413   1.125  1.00 46.84           C  \nATOM    793  O   LYS A 101      19.646   5.164   0.026  1.00 59.75           O  \nATOM    794  CB  LYS A 101      20.712   7.302   1.404  1.00 55.23           C  \nATOM    795  CG  LYS A 101      21.162   8.653   1.922  1.00 54.02           C  \nATOM    796  CD  LYS A 101      20.168   9.693   1.557  1.00 57.52           C  \nATOM    797  CE  LYS A 101      20.838  11.036   1.605  1.00 64.25           C  \nATOM    798  NZ  LYS A 101      19.913  11.825   0.750  1.00 64.04           N  \nATOM    799  N   GLY A 102      18.782   4.447   1.921  1.00 53.80           N  \nATOM    800  CA  GLY A 102      18.835   3.073   1.442  1.00 55.29           C  \nATOM    801  C   GLY A 102      20.267   2.567   1.297  1.00 61.86           C  \nATOM    802  O   GLY A 102      20.485   1.622   0.593  1.00 59.58           O  \nATOM    803  N   VAL A 103      21.256   3.199   1.941  1.00 67.35           N  \nATOM    804  CA  VAL A 103      22.633   2.632   1.992  1.00 63.43           C  \nATOM    805  C   VAL A 103      22.641   1.787   3.291  1.00 71.38           C  \nATOM    806  O   VAL A 103      22.223   2.285   4.341  1.00 71.26           O  \nATOM    807  CB  VAL A 103      23.745   3.747   1.937  1.00 59.27           C  \nATOM    808  CG1 VAL A 103      25.156   3.146   1.903  1.00 50.64           C  \nATOM    809  CG2 VAL A 103      23.575   4.635   0.700  1.00 47.69           C  \nATOM    810  N   LYS A 104      23.051   0.516   3.253  1.00 65.30           N  \nATOM    811  CA  LYS A 104      22.778  -0.323   4.439  1.00 68.64           C  \nATOM    812  C   LYS A 104      23.904  -0.482   5.514  1.00 55.99           C  \nATOM    813  O   LYS A 104      25.038  -0.832   5.202  1.00 58.24           O  \nATOM    814  CB  LYS A 104      22.075  -1.659   4.076  1.00 78.94           C  \nATOM    815  CG  LYS A 104      20.593  -1.469   3.726  1.00 87.64           C  \nATOM    816  CD  LYS A 104      19.665  -2.532   4.322  1.00 93.37           C  \nATOM    817  CE  LYS A 104      19.778  -3.868   3.597  1.00 98.56           C  \nATOM    818  NZ  LYS A 104      20.156  -3.682   2.166  1.00 89.09           N  \nATOM    819  N   PRO A 105      23.573  -0.183   6.789  1.00 55.12           N  \nATOM    820  CA  PRO A 105      24.544  -0.375   7.853  1.00 59.93           C  \nATOM    821  C   PRO A 105      25.075  -1.837   7.828  1.00 62.13           C  \nATOM    822  O   PRO A 105      24.343  -2.765   7.415  1.00 62.97           O  \nATOM    823  CB  PRO A 105      23.712  -0.123   9.120  1.00 59.53           C  \nATOM    824  CG  PRO A 105      22.629   0.822   8.678  1.00 60.25           C  \nATOM    825  CD  PRO A 105      22.271   0.319   7.310  1.00 54.46           C  \nATOM    826  N   ASP A 106      26.334  -2.025   8.229  1.00 54.45           N  \nATOM    827  CA  ASP A 106      26.968  -3.359   8.341  1.00 53.89           C  \nATOM    828  C   ASP A 106      27.557  -3.487   9.771  1.00 60.53           C  \nATOM    829  O   ASP A 106      27.300  -2.576  10.616  1.00 56.41           O  \nATOM    830  CB  ASP A 106      27.985  -3.569   7.228  1.00 50.50           C  \nATOM    831  CG  ASP A 106      29.242  -2.706   7.378  1.00 54.57           C  \nATOM    832  OD1 ASP A 106      29.528  -2.223   8.490  1.00 52.18           O  \nATOM    833  OD2 ASP A 106      29.988  -2.558   6.368  1.00 61.77           O  \nATOM    834  N   VAL A 107      28.307  -4.574  10.061  1.00 53.43           N  \nATOM    835  CA  VAL A 107      28.799  -4.803  11.414  1.00 52.34           C  \nATOM    836  C   VAL A 107      29.802  -3.740  11.845  1.00 49.24           C  \nATOM    837  O   VAL A 107      29.792  -3.321  13.028  1.00 50.42           O  \nATOM    838  CB  VAL A 107      29.368  -6.248  11.677  1.00 58.59           C  \nATOM    839  CG1 VAL A 107      30.677  -6.490  10.954  1.00 46.62           C  \nATOM    840  CG2 VAL A 107      29.640  -6.453  13.165  1.00 54.14           C  \nATOM    841  N   VAL A 108      30.652  -3.274  10.929  1.00 51.65           N  \nATOM    842  CA  VAL A 108      31.542  -2.143  11.295  1.00 59.56           C  \nATOM    843  C   VAL A 108      30.665  -0.897  11.739  1.00 47.53           C  \nATOM    844  O   VAL A 108      30.927  -0.231  12.758  1.00 49.87           O  \nATOM    845  CB  VAL A 108      32.559  -1.848  10.156  1.00 66.82           C  \nATOM    846  CG1 VAL A 108      33.386  -0.605  10.464  1.00 49.45           C  \nATOM    847  CG2 VAL A 108      33.464  -3.074   9.927  1.00 55.65           C  \nATOM    848  N   THR A 109      29.549  -0.685  11.044  1.00 45.36           N  \nATOM    849  CA  THR A 109      28.677   0.468  11.388  1.00 54.41           C  \nATOM    850  C   THR A 109      28.134   0.283  12.836  1.00 54.93           C  \nATOM    851  O   THR A 109      28.208   1.182  13.727  1.00 49.34           O  \nATOM    852  CB  THR A 109      27.488   0.748  10.383  1.00 50.41           C  \nATOM    853  OG1 THR A 109      27.821   0.459   8.985  1.00 46.71           O  \nATOM    854  CG2 THR A 109      26.940   2.213  10.600  1.00 49.57           C  \nATOM    855  N   TYR A 110      27.595  -0.901  13.096  1.00 50.95           N  \nATOM    856  CA  TYR A 110      27.003  -1.054  14.391  1.00 43.31           C  \nATOM    857  C   TYR A 110      28.050  -0.938  15.500  1.00 38.48           C  \nATOM    858  O   TYR A 110      27.773  -0.312  16.578  1.00 40.42           O  \nATOM    859  CB  TYR A 110      26.176  -2.315  14.411  1.00 47.23           C  \nATOM    860  CG  TYR A 110      24.856  -2.121  13.719  1.00 51.87           C  \nATOM    861  CD1 TYR A 110      23.774  -1.626  14.429  1.00 52.46           C  \nATOM    862  CD2 TYR A 110      24.671  -2.456  12.360  1.00 50.90           C  \nATOM    863  CE1 TYR A 110      22.542  -1.423  13.821  1.00 53.04           C  \nATOM    864  CE2 TYR A 110      23.437  -2.265  11.748  1.00 44.46           C  \nATOM    865  CZ  TYR A 110      22.376  -1.744  12.484  1.00 57.06           C  \nATOM    866  OH  TYR A 110      21.105  -1.534  11.931  1.00 70.19           O  \nATOM    867  N   ASN A 111      29.254  -1.474  15.286  1.00 40.78           N  \nATOM    868  CA  ASN A 111      30.241  -1.528  16.436  1.00 42.70           C  \nATOM    869  C   ASN A 111      30.718  -0.199  16.875  1.00 42.55           C  \nATOM    870  O   ASN A 111      30.926   0.078  18.096  1.00 41.40           O  \nATOM    871  CB  ASN A 111      31.490  -2.384  16.133  1.00 50.22           C  \nATOM    872  CG  ASN A 111      31.196  -3.914  16.184  1.00 60.55           C  \nATOM    873  OD1 ASN A 111      30.113  -4.341  16.600  1.00 43.65           O  \nATOM    874  ND2 ASN A 111      32.129  -4.723  15.703  1.00 54.33           N  \nATOM    875  N   THR A 112      30.928   0.691  15.901  1.00 45.76           N  \nATOM    876  CA  THR A 112      31.507   1.999  16.318  1.00 39.11           C  \nATOM    877  C   THR A 112      30.488   2.755  17.117  1.00 34.46           C  \nATOM    878  O   THR A 112      30.766   3.350  18.168  1.00 40.99           O  \nATOM    879  CB  THR A 112      31.792   2.865  15.063  1.00 52.35           C  \nATOM    880  OG1 THR A 112      32.004   1.977  13.958  1.00 50.83           O  \nATOM    881  CG2 THR A 112      33.032   3.742  15.338  1.00 54.07           C  \nATOM    882  N   LEU A 113      29.250   2.686  16.638  1.00 38.75           N  \nATOM    883  CA  LEU A 113      28.160   3.369  17.339  1.00 39.00           C  \nATOM    884  C   LEU A 113      27.934   2.743  18.716  1.00 36.14           C  \nATOM    885  O   LEU A 113      27.795   3.437  19.747  1.00 39.64           O  \nATOM    886  CB  LEU A 113      26.895   3.299  16.452  1.00 35.33           C  \nATOM    887  CG  LEU A 113      26.981   4.178  15.184  1.00 36.97           C  \nATOM    888  CD1 LEU A 113      25.802   3.855  14.279  1.00 35.26           C  \nATOM    889  CD2 LEU A 113      26.988   5.699  15.569  1.00 37.51           C  \nATOM    890  N   ILE A 114      27.915   1.413  18.772  1.00 45.28           N  \nATOM    891  CA  ILE A 114      27.657   0.733  20.094  1.00 40.22           C  \nATOM    892  C   ILE A 114      28.772   1.132  21.030  1.00 37.27           C  \nATOM    893  O   ILE A 114      28.592   1.569  22.219  1.00 38.39           O  \nATOM    894  CB  ILE A 114      27.506  -0.809  19.979  1.00 39.57           C  \nATOM    895  CG1 ILE A 114      26.153  -1.089  19.337  1.00 38.62           C  \nATOM    896  CG2 ILE A 114      27.505  -1.402  21.405  1.00 40.02           C  \nATOM    897  CD1 ILE A 114      26.080  -2.336  18.496  1.00 41.25           C  \nATOM    898  N   ASP A 115      29.945   1.056  20.479  1.00 42.02           N  \nATOM    899  CA  ASP A 115      31.037   1.436  21.316  1.00 45.57           C  \nATOM    900  C   ASP A 115      30.987   2.903  21.778  1.00 54.98           C  \nATOM    901  O   ASP A 115      31.176   3.199  23.000  1.00 50.11           O  \nATOM    902  CB  ASP A 115      32.318   1.229  20.563  1.00 54.32           C  \nATOM    903  CG  ASP A 115      33.469   1.649  21.401  1.00 70.68           C  \nATOM    904  OD1 ASP A 115      33.502   1.312  22.615  1.00 68.48           O  \nATOM    905  OD2 ASP A 115      34.303   2.384  20.877  1.00 99.00           O  \nATOM    906  N   GLY A 116      30.737   3.831  20.821  1.00 48.50           N  \nATOM    907  CA  GLY A 116      30.684   5.250  21.218  1.00 42.03           C  \nATOM    908  C   GLY A 116      29.627   5.404  22.267  1.00 47.58           C  \nATOM    909  O   GLY A 116      29.829   6.042  23.310  1.00 51.18           O  \nATOM    910  N   LEU A 117      28.488   4.763  22.048  1.00 42.30           N  \nATOM    911  CA  LEU A 117      27.393   4.953  23.001  1.00 43.07           C  \nATOM    912  C   LEU A 117      27.725   4.364  24.375  1.00 58.23           C  \nATOM    913  O   LEU A 117      27.407   4.947  25.424  1.00 54.02           O  \nATOM    914  CB  LEU A 117      26.115   4.298  22.487  1.00 42.34           C  \nATOM    915  CG  LEU A 117      25.406   4.955  21.306  1.00 52.91           C  \nATOM    916  CD1 LEU A 117      24.430   3.949  20.659  1.00 44.30           C  \nATOM    917  CD2 LEU A 117      24.680   6.196  21.845  1.00 46.50           C  \nATOM    918  N   ALA A 118      28.342   3.180  24.379  1.00 54.40           N  \nATOM    919  CA  ALA A 118      28.698   2.593  25.665  1.00 55.80           C  \nATOM    920  C   ALA A 118      29.705   3.497  26.401  1.00 56.77           C  \nATOM    921  O   ALA A 118      29.620   3.583  27.579  1.00 63.82           O  \nATOM    922  CB  ALA A 118      29.210   1.162  25.510  1.00 50.12           C  \nATOM    923  N   LYS A 119      30.602   4.207  25.728  1.00 64.02           N  \nATOM    924  CA  LYS A 119      31.316   5.343  26.385  1.00 71.23           C  \nATOM    925  C   LYS A 119      30.490   6.396  27.164  1.00 72.96           C  \nATOM    926  O   LYS A 119      30.841   6.714  28.284  1.00 86.07           O  \nATOM    927  CB  LYS A 119      32.251   6.051  25.416  1.00 73.90           C  \nATOM    928  CG  LYS A 119      33.715   5.706  25.618  1.00 78.35           C  \nATOM    929  CD  LYS A 119      33.858   4.336  26.243  1.00 83.39           C  \nATOM    930  CE  LYS A 119      33.519   3.252  25.243  1.00 79.21           C  \nATOM    931  NZ  LYS A 119      34.615   3.169  24.254  1.00 88.19           N  \nATOM    932  N   ALA A 120      29.430   6.965  26.596  1.00 70.30           N  \nATOM    933  CA  ALA A 120      28.506   7.783  27.391  1.00 69.06           C  \nATOM    934  C   ALA A 120      27.738   6.748  28.207  1.00 74.20           C  \nATOM    935  O   ALA A 120      28.002   5.581  28.055  1.00 84.80           O  \nATOM    936  CB  ALA A 120      27.582   8.548  26.486  1.00 68.08           C  \nATOM    937  N   GLY A 121      26.784   7.113  29.051  1.00 68.22           N  \nATOM    938  CA  GLY A 121      26.114   6.070  29.829  1.00 65.12           C  \nATOM    939  C   GLY A 121      25.064   5.365  29.002  1.00 74.33           C  \nATOM    940  O   GLY A 121      24.189   4.679  29.543  1.00 76.93           O  \nATOM    941  N   ARG A 122      25.179   5.508  27.681  1.00 67.55           N  \nATOM    942  CA  ARG A 122      24.069   5.278  26.778  1.00 64.46           C  \nATOM    943  C   ARG A 122      23.908   3.821  26.429  1.00 63.93           C  \nATOM    944  O   ARG A 122      23.719   3.372  25.260  1.00 55.39           O  \nATOM    945  CB  ARG A 122      24.158   6.235  25.601  1.00 60.79           C  \nATOM    946  CG  ARG A 122      24.276   7.641  26.154  1.00 64.55           C  \nATOM    947  CD  ARG A 122      24.371   8.700  25.093  1.00 58.49           C  \nATOM    948  NE  ARG A 122      23.198   8.730  24.229  1.00 53.28           N  \nATOM    949  CZ  ARG A 122      23.153   9.475  23.127  1.00 50.36           C  \nATOM    950  NH1 ARG A 122      24.201  10.190  22.842  1.00 46.71           N  \nATOM    951  NH2 ARG A 122      22.104   9.481  22.298  1.00 46.49           N  \nATOM    952  N   LEU A 123      23.850   3.082  27.517  1.00 64.00           N  \nATOM    953  CA  LEU A 123      23.896   1.667  27.454  1.00 57.27           C  \nATOM    954  C   LEU A 123      22.589   1.145  26.910  1.00 61.72           C  \nATOM    955  O   LEU A 123      22.559   0.186  26.127  1.00 66.19           O  \nATOM    956  CB  LEU A 123      24.211   1.180  28.855  1.00 62.14           C  \nATOM    957  CG  LEU A 123      24.931  -0.133  29.008  1.00 60.89           C  \nATOM    958  CD1 LEU A 123      26.122  -0.347  28.057  1.00 46.60           C  \nATOM    959  CD2 LEU A 123      25.340  -0.236  30.472  1.00 66.76           C  \nATOM    960  N   GLU A 124      21.483   1.790  27.242  1.00 58.93           N  \nATOM    961  CA  GLU A 124      20.230   1.247  26.691  1.00 58.93           C  \nATOM    962  C   GLU A 124      20.166   1.402  25.161  1.00 61.45           C  \nATOM    963  O   GLU A 124      19.666   0.514  24.432  1.00 58.33           O  \nATOM    964  CB  GLU A 124      19.012   1.867  27.385  1.00 60.80           C  \nATOM    965  CG  GLU A 124      19.001   1.620  28.891  1.00 77.26           C  \nATOM    966  CD  GLU A 124      18.893   0.122  29.223  1.00 81.22           C  \nATOM    967  OE1 GLU A 124      17.811  -0.453  29.000  1.00 77.23           O  \nATOM    968  OE2 GLU A 124      19.892  -0.500  29.677  1.00 72.00           O  \nATOM    969  N   GLU A 125      20.665   2.539  24.671  1.00 57.89           N  \nATOM    970  CA  GLU A 125      20.731   2.750  23.218  1.00 57.59           C  \nATOM    971  C   GLU A 125      21.803   1.853  22.607  1.00 43.76           C  \nATOM    972  O   GLU A 125      21.628   1.341  21.516  1.00 44.80           O  \nATOM    973  CB  GLU A 125      21.051   4.216  22.896  1.00 52.92           C  \nATOM    974  CG  GLU A 125      20.018   5.186  23.431  1.00 58.92           C  \nATOM    975  CD  GLU A 125      20.458   6.603  23.198  1.00 60.57           C  \nATOM    976  OE1 GLU A 125      20.487   7.024  22.024  1.00 49.91           O  \nATOM    977  OE2 GLU A 125      20.846   7.251  24.170  1.00 54.87           O  \nATOM    978  N   ALA A 126      22.915   1.670  23.308  1.00 45.09           N  \nATOM    979  CA  ALA A 126      23.929   0.662  22.862  1.00 47.85           C  \nATOM    980  C   ALA A 126      23.253  -0.726  22.614  1.00 48.33           C  \nATOM    981  O   ALA A 126      23.410  -1.352  21.524  1.00 45.66           O  \nATOM    982  CB  ALA A 126      25.050   0.560  23.879  1.00 40.81           C  \nATOM    983  N   LEU A 127      22.378  -1.118  23.552  1.00 48.30           N  \nATOM    984  CA  LEU A 127      21.717  -2.436  23.516  1.00 53.59           C  \nATOM    985  C   LEU A 127      20.619  -2.462  22.519  1.00 59.17           C  \nATOM    986  O   LEU A 127      20.366  -3.465  21.843  1.00 60.17           O  \nATOM    987  CB  LEU A 127      21.169  -2.792  24.920  1.00 55.57           C  \nATOM    988  CG  LEU A 127      22.336  -3.202  25.828  1.00 55.57           C  \nATOM    989  CD1 LEU A 127      22.134  -2.853  27.288  1.00 53.67           C  \nATOM    990  CD2 LEU A 127      22.612  -4.681  25.627  1.00 59.07           C  \nATOM    991  N   GLN A 128      19.933  -1.336  22.422  1.00 62.93           N  \nATOM    992  CA  GLN A 128      18.918  -1.218  21.415  1.00 61.47           C  \nATOM    993  C   GLN A 128      19.558  -1.383  20.048  1.00 52.70           C  \nATOM    994  O   GLN A 128      18.993  -1.986  19.153  1.00 57.95           O  \nATOM    995  CB  GLN A 128      18.290   0.148  21.561  1.00 75.17           C  \nATOM    996  CG  GLN A 128      16.989   0.344  20.843  1.00 81.47           C  \nATOM    997  CD  GLN A 128      16.529   1.772  21.013  1.00 99.74           C  \nATOM    998  OE1 GLN A 128      17.090   2.530  21.822  1.00101.19           O  \nATOM    999  NE2 GLN A 128      15.520   2.160  20.248  1.00104.22           N  \nATOM   1000  N   LEU A 129      20.762  -0.862  19.869  1.00 47.87           N  \nATOM   1001  CA  LEU A 129      21.344  -0.949  18.542  1.00 45.89           C  \nATOM   1002  C   LEU A 129      21.867  -2.367  18.308  1.00 44.83           C  \nATOM   1003  O   LEU A 129      21.813  -2.924  17.214  1.00 42.79           O  \nATOM   1004  CB  LEU A 129      22.484   0.072  18.490  1.00 47.02           C  \nATOM   1005  CG  LEU A 129      22.670   0.869  17.226  1.00 54.28           C  \nATOM   1006  CD1 LEU A 129      21.347   1.593  16.888  1.00 50.17           C  \nATOM   1007  CD2 LEU A 129      23.780   1.879  17.464  1.00 50.37           C  \nATOM   1008  N   PHE A 130      22.452  -2.934  19.342  1.00 46.12           N  \nATOM   1009  CA  PHE A 130      22.885  -4.351  19.286  1.00 52.80           C  \nATOM   1010  C   PHE A 130      21.749  -5.299  18.889  1.00 46.42           C  \nATOM   1011  O   PHE A 130      21.903  -6.168  18.025  1.00 53.45           O  \nATOM   1012  CB  PHE A 130      23.412  -4.753  20.665  1.00 53.03           C  \nATOM   1013  CG  PHE A 130      23.895  -6.182  20.752  1.00 56.53           C  \nATOM   1014  CD1 PHE A 130      25.034  -6.598  20.046  1.00 54.98           C  \nATOM   1015  CD2 PHE A 130      23.243  -7.103  21.589  1.00 64.14           C  \nATOM   1016  CE1 PHE A 130      25.507  -7.914  20.170  1.00 70.12           C  \nATOM   1017  CE2 PHE A 130      23.705  -8.423  21.708  1.00 65.63           C  \nATOM   1018  CZ  PHE A 130      24.841  -8.836  20.995  1.00 49.39           C  \nATOM   1019  N   GLN A 131      20.600  -5.105  19.515  1.00 51.45           N  \nATOM   1020  CA  GLN A 131      19.346  -5.744  19.097  1.00 63.31           C  \nATOM   1021  C   GLN A 131      18.996  -5.503  17.632  1.00 74.40           C  \nATOM   1022  O   GLN A 131      18.714  -6.440  16.867  1.00 83.59           O  \nATOM   1023  CB  GLN A 131      18.197  -5.251  19.976  1.00 61.55           C  \nATOM   1024  CG  GLN A 131      17.229  -6.369  20.361  1.00 86.81           C  \nATOM   1025  CD  GLN A 131      17.934  -7.726  20.545  1.00101.57           C  \nATOM   1026  OE1 GLN A 131      18.685  -7.941  21.514  1.00100.57           O  \nATOM   1027  NE2 GLN A 131      17.709  -8.643  19.593  1.00 96.76           N  \nATOM   1028  N   GLU A 132      19.013  -4.234  17.242  1.00 69.60           N  \nATOM   1029  CA  GLU A 132      18.686  -3.876  15.898  1.00 62.61           C  \nATOM   1030  C   GLU A 132      19.661  -4.542  14.964  1.00 57.06           C  \nATOM   1031  O   GLU A 132      19.299  -5.032  13.900  1.00 55.47           O  \nATOM   1032  CB  GLU A 132      18.772  -2.353  15.739  1.00 67.63           C  \nATOM   1033  CG  GLU A 132      18.697  -1.888  14.292  1.00 63.91           C  \nATOM   1034  CD  GLU A 132      18.774  -0.371  14.169  1.00 71.76           C  \nATOM   1035  OE1 GLU A 132      18.187   0.326  15.018  1.00 66.70           O  \nATOM   1036  OE2 GLU A 132      19.446   0.113  13.233  1.00 63.06           O  \nATOM   1037  N   MET A 133      20.923  -4.512  15.333  1.00 56.31           N  \nATOM   1038  CA  MET A 133      21.952  -5.107  14.486  1.00 66.70           C  \nATOM   1039  C   MET A 133      21.524  -6.516  14.062  1.00 78.17           C  \nATOM   1040  O   MET A 133      21.530  -6.840  12.885  1.00 85.94           O  \nATOM   1041  CB  MET A 133      23.225  -5.200  15.284  1.00 59.29           C  \nATOM   1042  CG  MET A 133      24.426  -5.549  14.470  1.00 62.02           C  \nATOM   1043  SD  MET A 133      25.574  -5.532  15.833  1.00 60.32           S  \nATOM   1044  CE  MET A 133      27.036  -5.924  14.884  1.00 52.84           C  \nATOM   1045  N   LYS A 134      21.119  -7.329  15.035  1.00 77.93           N  \nATOM   1046  CA  LYS A 134      20.725  -8.713  14.770  1.00 82.66           C  \nATOM   1047  C   LYS A 134      19.493  -8.820  13.882  1.00 86.35           C  \nATOM   1048  O   LYS A 134      19.506  -9.554  12.895  1.00 85.56           O  \nATOM   1049  CB  LYS A 134      20.470  -9.464  16.064  1.00 67.78           C  \nATOM   1050  CG  LYS A 134      21.720  -9.799  16.819  1.00 65.67           C  \nATOM   1051  CD  LYS A 134      21.281  -9.852  18.255  1.00 69.38           C  \nATOM   1052  CE  LYS A 134      22.355 -10.388  19.158  1.00 70.47           C  \nATOM   1053  NZ  LYS A 134      21.724 -10.259  20.497  1.00 81.83           N  \nATOM   1054  N   GLU A 135      18.433  -8.093  14.220  1.00 84.91           N  \nATOM   1055  CA  GLU A 135      17.216  -8.182  13.434  1.00 78.86           C  \nATOM   1056  C   GLU A 135      17.339  -7.706  11.978  1.00 79.42           C  \nATOM   1057  O   GLU A 135      16.473  -8.015  11.172  1.00 94.36           O  \nATOM   1058  CB  GLU A 135      16.060  -7.492  14.135  1.00 80.70           C  \nATOM   1059  CG  GLU A 135      15.996  -6.003  13.882  1.00 98.27           C  \nATOM   1060  CD  GLU A 135      14.942  -5.329  14.728  1.00108.96           C  \nATOM   1061  OE1 GLU A 135      14.607  -5.888  15.798  1.00110.13           O  \nATOM   1062  OE2 GLU A 135      14.457  -4.243  14.324  1.00112.20           O  \nATOM   1063  N   LYS A 136      18.377  -6.940  11.644  1.00 76.13           N  \nATOM   1064  CA  LYS A 136      18.645  -6.570  10.243  1.00 77.80           C  \nATOM   1065  C   LYS A 136      19.687  -7.523   9.714  1.00 85.52           C  \nATOM   1066  O   LYS A 136      20.440  -7.197   8.797  1.00 97.47           O  \nATOM   1067  CB  LYS A 136      19.086  -5.093  10.089  1.00 80.03           C  \nATOM   1068  CG  LYS A 136      18.034  -4.120  10.630  1.00 82.11           C  \nATOM   1069  CD  LYS A 136      18.230  -2.679  10.188  1.00 90.67           C  \nATOM   1070  CE  LYS A 136      17.100  -1.828  10.752  1.00 90.40           C  \nATOM   1071  NZ  LYS A 136      17.018  -0.520  10.053  1.00 94.25           N  \nATOM   1072  N   GLY A 137      19.739  -8.697  10.338  1.00 79.87           N  \nATOM   1073  CA  GLY A 137      20.608  -9.784   9.932  1.00 72.82           C  \nATOM   1074  C   GLY A 137      22.052  -9.410   9.835  1.00 72.01           C  \nATOM   1075  O   GLY A 137      22.788 -10.013   9.071  1.00 69.00           O  \nATOM   1076  N   VAL A 138      22.473  -8.406  10.596  1.00 77.27           N  \nATOM   1077  CA  VAL A 138      23.917  -8.056  10.694  1.00 76.72           C  \nATOM   1078  C   VAL A 138      24.493  -8.723  11.974  1.00 82.77           C  \nATOM   1079  O   VAL A 138      23.831  -8.726  13.033  1.00 83.90           O  \nATOM   1080  CB  VAL A 138      24.167  -6.504  10.645  1.00 72.57           C  \nATOM   1081  CG1 VAL A 138      25.633  -6.193  10.837  1.00 59.19           C  \nATOM   1082  CG2 VAL A 138      23.689  -5.881   9.323  1.00 60.67           C  \nATOM   1083  N   LYS A 139      25.734  -9.217  11.868  1.00 86.60           N  \nATOM   1084  CA  LYS A 139      26.328 -10.277  12.719  1.00 91.66           C  \nATOM   1085  C   LYS A 139      27.474  -9.771  13.672  1.00 87.23           C  \nATOM   1086  O   LYS A 139      28.537  -9.379  13.174  1.00100.37           O  \nATOM   1087  CB  LYS A 139      26.887 -11.328  11.723  1.00 98.07           C  \nATOM   1088  CG  LYS A 139      27.115 -12.763  12.174  1.00 88.15           C  \nATOM   1089  CD  LYS A 139      26.893 -13.702  10.980  1.00 91.26           C  \nATOM   1090  CE  LYS A 139      28.116 -14.556  10.640  1.00 95.94           C  \nATOM   1091  NZ  LYS A 139      28.380 -15.670  11.600  1.00 85.90           N  \nATOM   1092  N   PRO A 140      27.282  -9.799  15.029  1.00 69.30           N  \nATOM   1093  CA  PRO A 140      28.279  -9.325  16.072  1.00 66.40           C  \nATOM   1094  C   PRO A 140      29.735  -9.925  16.163  1.00 63.41           C  \nATOM   1095  O   PRO A 140      30.135 -10.684  15.304  1.00 74.43           O  \nATOM   1096  CB  PRO A 140      27.520  -9.526  17.377  1.00 67.44           C  \nATOM   1097  CG  PRO A 140      26.058  -9.418  16.991  1.00 71.77           C  \nATOM   1098  CD  PRO A 140      25.938 -10.016  15.620  1.00 70.84           C  \nATOM   1099  N   ASP A 141      30.518  -9.577  17.195  1.00 66.61           N  \nATOM   1100  CA  ASP A 141      31.957  -9.960  17.321  1.00 63.39           C  \nATOM   1101  C   ASP A 141      32.671  -9.514  18.675  1.00 72.39           C  \nATOM   1102  O   ASP A 141      32.064  -8.889  19.540  1.00 73.44           O  \nATOM   1103  CB  ASP A 141      32.751  -9.605  16.024  1.00 72.02           C  \nATOM   1104  CG  ASP A 141      33.648  -8.343  16.159  1.00 93.06           C  \nATOM   1105  OD1 ASP A 141      33.311  -7.396  16.914  1.00 79.74           O  \nATOM   1106  OD2 ASP A 141      34.722  -8.302  15.502  1.00 92.82           O  \nATOM   1107  N   VAL A 142      33.943  -9.874  18.857  1.00 76.79           N  \nATOM   1108  CA  VAL A 142      34.614  -9.787  20.169  1.00 81.27           C  \nATOM   1109  C   VAL A 142      34.625  -8.368  20.687  1.00 81.64           C  \nATOM   1110  O   VAL A 142      34.527  -8.115  21.907  1.00 71.59           O  \nATOM   1111  CB  VAL A 142      36.108 -10.213  20.116  1.00 73.78           C  \nATOM   1112  CG1 VAL A 142      36.558 -10.736  21.478  1.00 77.87           C  \nATOM   1113  CG2 VAL A 142      36.369 -11.204  18.990  1.00 76.47           C  \nATOM   1114  N   VAL A 143      34.824  -7.458  19.736  1.00 83.66           N  \nATOM   1115  CA  VAL A 143      34.870  -6.021  19.982  1.00 69.30           C  \nATOM   1116  C   VAL A 143      33.463  -5.639  20.494  1.00 56.72           C  \nATOM   1117  O   VAL A 143      33.335  -4.961  21.529  1.00 56.93           O  \nATOM   1118  CB  VAL A 143      35.274  -5.255  18.674  1.00 63.78           C  \nATOM   1119  CG1 VAL A 143      35.943  -6.192  17.659  1.00 66.08           C  \nATOM   1120  CG2 VAL A 143      34.096  -4.594  17.995  1.00 58.15           C  \nATOM   1121  N   THR A 144      32.432  -6.144  19.795  1.00 50.81           N  \nATOM   1122  CA  THR A 144      31.030  -5.735  20.050  1.00 63.44           C  \nATOM   1123  C   THR A 144      30.679  -6.154  21.524  1.00 65.88           C  \nATOM   1124  O   THR A 144      30.760  -5.322  22.518  1.00 56.16           O  \nATOM   1125  CB  THR A 144      30.043  -6.003  18.798  1.00 55.75           C  \nATOM   1126  OG1 THR A 144      28.758  -6.527  19.108  1.00 63.13           O  \nATOM   1127  CG2 THR A 144      30.580  -6.808  17.722  1.00 48.72           C  \nATOM   1128  N   TYR A 145      30.459  -7.462  21.658  1.00 67.66           N  \nATOM   1129  CA  TYR A 145      30.517  -8.169  22.944  1.00 46.51           C  \nATOM   1130  C   TYR A 145      31.346  -7.544  24.043  1.00 40.27           C  \nATOM   1131  O   TYR A 145      30.766  -7.135  25.079  1.00 47.31           O  \nATOM   1132  CB  TYR A 145      30.779  -9.658  22.695  1.00 50.77           C  \nATOM   1133  CG  TYR A 145      29.566 -10.370  22.092  1.00 41.40           C  \nATOM   1134  CD1 TYR A 145      28.396 -10.549  22.829  1.00 45.00           C  \nATOM   1135  CD2 TYR A 145      29.617 -10.928  20.854  1.00 38.29           C  \nATOM   1136  CE1 TYR A 145      27.293 -11.216  22.283  1.00 44.94           C  \nATOM   1137  CE2 TYR A 145      28.518 -11.560  20.283  1.00 38.15           C  \nATOM   1138  CZ  TYR A 145      27.361 -11.703  20.985  1.00 47.04           C  \nATOM   1139  OH  TYR A 145      26.254 -12.349  20.414  1.00 54.27           O  \nATOM   1140  N   ASN A 146      32.644  -7.287  23.871  1.00 42.62           N  \nATOM   1141  CA  ASN A 146      33.372  -6.719  25.024  1.00 39.43           C  \nATOM   1142  C   ASN A 146      32.981  -5.324  25.512  1.00 55.69           C  \nATOM   1143  O   ASN A 146      33.240  -4.911  26.711  1.00 49.59           O  \nATOM   1144  CB  ASN A 146      34.902  -6.618  24.725  1.00 54.08           C  \nATOM   1145  CG  ASN A 146      35.686  -7.945  24.929  1.00 67.72           C  \nATOM   1146  OD1 ASN A 146      35.129  -9.097  24.915  1.00 49.52           O  \nATOM   1147  ND2 ASN A 146      37.015  -7.788  25.056  1.00 56.40           N  \nATOM   1148  N   THR A 147      32.486  -4.504  24.586  1.00 57.05           N  \nATOM   1149  CA  THR A 147      32.233  -3.125  25.038  1.00 57.58           C  \nATOM   1150  C   THR A 147      30.894  -3.183  25.731  1.00 42.09           C  \nATOM   1151  O   THR A 147      30.719  -2.630  26.801  1.00 51.07           O  \nATOM   1152  CB  THR A 147      32.346  -2.095  23.893  1.00 65.00           C  \nATOM   1153  OG1 THR A 147      31.875  -2.690  22.686  1.00 61.09           O  \nATOM   1154  CG2 THR A 147      33.812  -1.805  23.652  1.00 68.18           C  \nATOM   1155  N   LEU A 148      29.971  -3.944  25.152  1.00 49.00           N  \nATOM   1156  CA  LEU A 148      28.700  -4.216  25.878  1.00 44.10           C  \nATOM   1157  C   LEU A 148      28.916  -4.816  27.268  1.00 44.36           C  \nATOM   1158  O   LEU A 148      28.382  -4.307  28.315  1.00 39.40           O  \nATOM   1159  CB  LEU A 148      27.781  -5.036  24.986  1.00 42.74           C  \nATOM   1160  CG  LEU A 148      27.283  -4.250  23.762  1.00 48.85           C  \nATOM   1161  CD1 LEU A 148      26.508  -5.183  22.867  1.00 46.37           C  \nATOM   1162  CD2 LEU A 148      26.369  -3.100  24.189  1.00 41.82           C  \nATOM   1163  N   ILE A 149      29.804  -5.821  27.304  1.00 45.85           N  \nATOM   1164  CA  ILE A 149      30.079  -6.541  28.549  1.00 44.83           C  \nATOM   1165  C   ILE A 149      30.729  -5.587  29.491  1.00 44.18           C  \nATOM   1166  O   ILE A 149      30.326  -5.477  30.690  1.00 45.19           O  \nATOM   1167  CB  ILE A 149      30.937  -7.825  28.311  1.00 38.99           C  \nATOM   1168  CG1 ILE A 149      30.016  -8.899  27.736  1.00 44.14           C  \nATOM   1169  CG2 ILE A 149      31.572  -8.243  29.630  1.00 43.99           C  \nATOM   1170  CD1 ILE A 149      30.744  -9.974  26.940  1.00 41.50           C  \nATOM   1171  N   ASP A 150      31.708  -4.827  28.975  1.00 50.31           N  \nATOM   1172  CA  ASP A 150      32.341  -3.877  29.899  1.00 47.60           C  \nATOM   1173  C   ASP A 150      31.350  -2.814  30.280  1.00 48.76           C  \nATOM   1174  O   ASP A 150      31.227  -2.467  31.480  1.00 53.75           O  \nATOM   1175  CB  ASP A 150      33.600  -3.242  29.346  1.00 67.08           C  \nATOM   1176  CG  ASP A 150      34.168  -2.179  30.315  1.00 72.48           C  \nATOM   1177  OD1 ASP A 150      34.636  -2.528  31.418  1.00 72.20           O  \nATOM   1178  OD2 ASP A 150      34.096  -0.982  30.002  1.00 81.59           O  \nATOM   1179  N   GLY A 151      30.576  -2.321  29.296  1.00 44.68           N  \nATOM   1180  CA  GLY A 151      29.533  -1.331  29.668  1.00 47.02           C  \nATOM   1181  C   GLY A 151      28.663  -1.848  30.813  1.00 51.61           C  \nATOM   1182  O   GLY A 151      28.502  -1.202  31.883  1.00 49.52           O  \nATOM   1183  N   LEU A 152      28.079  -3.037  30.613  1.00 46.27           N  \nATOM   1184  CA  LEU A 152      27.118  -3.520  31.595  1.00 40.09           C  \nATOM   1185  C   LEU A 152      27.742  -3.755  32.931  1.00 40.65           C  \nATOM   1186  O   LEU A 152      27.155  -3.421  33.985  1.00 49.88           O  \nATOM   1187  CB  LEU A 152      26.471  -4.818  31.087  1.00 43.39           C  \nATOM   1188  CG  LEU A 152      25.563  -4.600  29.931  1.00 45.66           C  \nATOM   1189  CD1 LEU A 152      25.519  -5.853  29.084  1.00 40.64           C  \nATOM   1190  CD2 LEU A 152      24.197  -4.203  30.549  1.00 42.52           C  \nATOM   1191  N   ALA A 153      28.911  -4.390  32.926  1.00 47.42           N  \nATOM   1192  CA  ALA A 153      29.622  -4.582  34.192  1.00 55.15           C  \nATOM   1193  C   ALA A 153      29.816  -3.204  34.867  1.00 57.77           C  \nATOM   1194  O   ALA A 153      29.373  -3.001  35.997  1.00 60.54           O  \nATOM   1195  CB  ALA A 153      30.952  -5.303  33.976  1.00 47.34           C  \nATOM   1196  N   LYS A 154      30.385  -2.229  34.158  1.00 67.01           N  \nATOM   1197  CA  LYS A 154      30.598  -0.887  34.788  1.00 70.43           C  \nATOM   1198  C   LYS A 154      29.350  -0.359  35.463  1.00 70.70           C  \nATOM   1199  O   LYS A 154      29.455   0.192  36.553  1.00 65.60           O  \nATOM   1200  CB  LYS A 154      31.145   0.149  33.811  1.00 71.19           C  \nATOM   1201  CG  LYS A 154      32.581  -0.119  33.379  1.00 80.19           C  \nATOM   1202  CD  LYS A 154      33.561   0.102  34.522  1.00 83.41           C  \nATOM   1203  CE  LYS A 154      34.944   0.434  33.982  1.00 80.06           C  \nATOM   1204  NZ  LYS A 154      35.727  -0.788  33.674  1.00 90.15           N  \nATOM   1205  N   ALA A 155      28.181  -0.581  34.837  1.00 64.26           N  \nATOM   1206  CA  ALA A 155      26.888  -0.122  35.332  1.00 52.55           C  \nATOM   1207  C   ALA A 155      26.186  -1.035  36.327  1.00 61.00           C  \nATOM   1208  O   ALA A 155      24.952  -1.052  36.385  1.00 72.29           O  \nATOM   1209  CB  ALA A 155      25.956   0.113  34.155  1.00 50.89           C  \nATOM   1210  N   GLY A 156      26.919  -1.829  37.091  1.00 62.70           N  \nATOM   1211  CA  GLY A 156      26.242  -2.711  38.039  1.00 58.00           C  \nATOM   1212  C   GLY A 156      25.531  -3.931  37.448  1.00 67.84           C  \nATOM   1213  O   GLY A 156      25.113  -4.821  38.195  1.00 69.92           O  \nATOM   1214  N   ARG A 157      25.413  -4.006  36.118  1.00 57.14           N  \nATOM   1215  CA  ARG A 157      24.542  -4.997  35.497  1.00 49.41           C  \nATOM   1216  C   ARG A 157      25.268  -6.293  35.085  1.00 49.69           C  \nATOM   1217  O   ARG A 157      25.404  -6.660  33.916  1.00 47.76           O  \nATOM   1218  CB  ARG A 157      23.782  -4.348  34.375  1.00 46.56           C  \nATOM   1219  CG  ARG A 157      23.096  -3.092  34.883  1.00 46.60           C  \nATOM   1220  CD  ARG A 157      22.695  -2.208  33.731  1.00 52.02           C  \nATOM   1221  NE  ARG A 157      21.776  -2.870  32.826  1.00 48.38           N  \nATOM   1222  CZ  ARG A 157      21.203  -2.248  31.805  1.00 49.39           C  \nATOM   1223  NH1 ARG A 157      20.358  -2.872  30.978  1.00 38.63           N  \nATOM   1224  NH2 ARG A 157      21.496  -0.976  31.604  1.00 51.97           N  \nATOM   1225  N   LEU A 158      25.717  -6.958  36.127  1.00 52.80           N  \nATOM   1226  CA  LEU A 158      26.549  -8.091  36.076  1.00 51.14           C  \nATOM   1227  C   LEU A 158      25.808  -9.239  35.475  1.00 43.40           C  \nATOM   1228  O   LEU A 158      26.355  -9.924  34.617  1.00 48.53           O  \nATOM   1229  CB  LEU A 158      26.963  -8.461  37.498  1.00 53.78           C  \nATOM   1230  CG  LEU A 158      28.337  -9.101  37.571  1.00 53.23           C  \nATOM   1231  CD1 LEU A 158      29.252  -8.366  36.588  1.00 52.87           C  \nATOM   1232  CD2 LEU A 158      28.850  -8.979  39.006  1.00 52.99           C  \nATOM   1233  N   GLU A 159      24.564  -9.444  35.903  1.00 43.72           N  \nATOM   1234  CA  GLU A 159      23.755 -10.545  35.359  1.00 42.33           C  \nATOM   1235  C   GLU A 159      23.629 -10.458  33.856  1.00 41.66           C  \nATOM   1236  O   GLU A 159      23.844 -11.450  33.134  1.00 36.48           O  \nATOM   1237  CB  GLU A 159      22.357 -10.600  36.011  1.00 41.97           C  \nATOM   1238  CG  GLU A 159      22.341 -11.234  37.389  1.00 58.03           C  \nATOM   1239  CD  GLU A 159      23.168 -12.526  37.472  1.00 57.75           C  \nATOM   1240  OE1 GLU A 159      24.292 -12.468  37.996  1.00 61.48           O  \nATOM   1241  OE2 GLU A 159      22.715 -13.592  36.991  1.00 49.61           O  \nATOM   1242  N   GLU A 160      23.300  -9.273  33.340  1.00 39.57           N  \nATOM   1243  CA  GLU A 160      23.219  -9.180  31.867  1.00 37.78           C  \nATOM   1244  C   GLU A 160      24.582  -9.268  31.218  1.00 36.93           C  \nATOM   1245  O   GLU A 160      24.698  -9.755  30.088  1.00 36.71           O  \nATOM   1246  CB  GLU A 160      22.530  -7.910  31.393  1.00 43.70           C  \nATOM   1247  CG  GLU A 160      21.121  -7.750  31.925  1.00 55.82           C  \nATOM   1248  CD  GLU A 160      20.745  -6.291  31.978  1.00 63.30           C  \nATOM   1249  OE1 GLU A 160      20.898  -5.678  33.050  1.00 58.65           O  \nATOM   1250  OE2 GLU A 160      20.407  -5.757  30.907  1.00 59.80           O  \nATOM   1251  N   ALA A 161      25.624  -8.772  31.879  1.00 35.93           N  \nATOM   1252  CA  ALA A 161      26.971  -8.924  31.240  1.00 35.73           C  \nATOM   1253  C   ALA A 161      27.338 -10.434  31.036  1.00 33.17           C  \nATOM   1254  O   ALA A 161      27.845 -10.848  29.993  1.00 34.19           O  \nATOM   1255  CB  ALA A 161      28.003  -8.253  32.148  1.00 35.79           C  \nATOM   1256  N   LEU A 162      27.152 -11.211  32.113  1.00 34.01           N  \nATOM   1257  CA  LEU A 162      27.432 -12.668  32.173  1.00 36.60           C  \nATOM   1258  C   LEU A 162      26.613 -13.414  31.153  1.00 34.27           C  \nATOM   1259  O   LEU A 162      27.120 -14.216  30.387  1.00 40.98           O  \nATOM   1260  CB  LEU A 162      27.051 -13.209  33.548  1.00 34.72           C  \nATOM   1261  CG  LEU A 162      28.104 -12.808  34.566  1.00 44.05           C  \nATOM   1262  CD1 LEU A 162      27.458 -13.144  35.935  1.00 40.90           C  \nATOM   1263  CD2 LEU A 162      29.374 -13.640  34.241  1.00 36.84           C  \nATOM   1264  N   GLN A 163      25.365 -12.988  31.044  1.00 37.94           N  \nATOM   1265  CA  GLN A 163      24.500 -13.529  30.031  1.00 43.37           C  \nATOM   1266  C   GLN A 163      24.943 -13.238  28.578  1.00 45.15           C  \nATOM   1267  O   GLN A 163      24.917 -14.137  27.717  1.00 43.18           O  \nATOM   1268  CB  GLN A 163      23.115 -13.019  30.331  1.00 44.80           C  \nATOM   1269  CG  GLN A 163      22.238 -12.963  29.141  1.00 59.24           C  \nATOM   1270  CD  GLN A 163      20.804 -12.848  29.599  1.00 79.38           C  \nATOM   1271  OE1 GLN A 163      20.545 -12.578  30.787  1.00 88.43           O  \nATOM   1272  NE2 GLN A 163      19.864 -13.078  28.683  1.00 68.52           N  \nATOM   1273  N   LEU A 164      25.409 -12.020  28.285  1.00 39.01           N  \nATOM   1274  CA  LEU A 164      25.988 -11.769  26.934  1.00 37.24           C  \nATOM   1275  C   LEU A 164      27.259 -12.577  26.715  1.00 32.88           C  \nATOM   1276  O   LEU A 164      27.567 -12.984  25.583  1.00 32.98           O  \nATOM   1277  CB  LEU A 164      26.439 -10.280  26.724  1.00 35.40           C  \nATOM   1278  CG  LEU A 164      25.418  -9.255  26.351  1.00 50.37           C  \nATOM   1279  CD1 LEU A 164      26.143  -7.950  25.955  1.00 48.73           C  \nATOM   1280  CD2 LEU A 164      24.465  -9.800  25.282  1.00 52.16           C  \nATOM   1281  N   PHE A 165      28.073 -12.657  27.770  1.00 30.15           N  \nATOM   1282  CA  PHE A 165      29.240 -13.544  27.766  1.00 34.16           C  \nATOM   1283  C   PHE A 165      28.896 -14.979  27.296  1.00 33.25           C  \nATOM   1284  O   PHE A 165      29.570 -15.521  26.408  1.00 34.09           O  \nATOM   1285  CB  PHE A 165      29.787 -13.588  29.158  1.00 37.41           C  \nATOM   1286  CG  PHE A 165      30.989 -14.446  29.312  1.00 38.01           C  \nATOM   1287  CD1 PHE A 165      32.130 -14.165  28.637  1.00 39.44           C  \nATOM   1288  CD2 PHE A 165      30.954 -15.552  30.184  1.00 41.41           C  \nATOM   1289  CE1 PHE A 165      33.268 -14.963  28.786  1.00 40.39           C  \nATOM   1290  CE2 PHE A 165      32.074 -16.346  30.352  1.00 45.71           C  \nATOM   1291  CZ  PHE A 165      33.241 -16.045  29.649  1.00 41.93           C  \nATOM   1292  N   GLN A 166      27.813 -15.548  27.799  1.00 34.25           N  \nATOM   1293  CA  GLN A 166      27.488 -16.970  27.408  1.00 38.14           C  \nATOM   1294  C   GLN A 166      27.025 -16.987  25.940  1.00 40.51           C  \nATOM   1295  O   GLN A 166      27.433 -17.863  25.090  1.00 39.87           O  \nATOM   1296  CB  GLN A 166      26.414 -17.584  28.355  1.00 35.26           C  \nATOM   1297  CG  GLN A 166      26.839 -17.822  29.863  1.00 47.73           C  \nATOM   1298  CD  GLN A 166      28.223 -18.562  30.058  1.00 50.73           C  \nATOM   1299  OE1 GLN A 166      28.918 -18.870  29.102  1.00 61.24           O  \nATOM   1300  NE2 GLN A 166      28.607 -18.812  31.280  1.00 45.26           N  \nATOM   1301  N   GLU A 167      26.232 -15.973  25.601  1.00 42.47           N  \nATOM   1302  CA  GLU A 167      25.727 -15.813  24.214  1.00 43.98           C  \nATOM   1303  C   GLU A 167      26.890 -15.707  23.308  1.00 42.58           C  \nATOM   1304  O   GLU A 167      26.998 -16.387  22.295  1.00 44.50           O  \nATOM   1305  CB  GLU A 167      24.884 -14.560  24.076  1.00 43.55           C  \nATOM   1306  CG  GLU A 167      24.228 -14.417  22.703  1.00 51.49           C  \nATOM   1307  CD  GLU A 167      23.452 -13.100  22.602  1.00 57.06           C  \nATOM   1308  OE1 GLU A 167      22.652 -12.837  23.528  1.00 61.75           O  \nATOM   1309  OE2 GLU A 167      23.707 -12.303  21.667  1.00 60.90           O  \nATOM   1310  N   MET A 168      27.832 -14.890  23.715  1.00 41.51           N  \nATOM   1311  CA  MET A 168      29.042 -14.726  22.905  1.00 45.80           C  \nATOM   1312  C   MET A 168      29.712 -16.134  22.570  1.00 53.71           C  \nATOM   1313  O   MET A 168      30.115 -16.427  21.413  1.00 42.81           O  \nATOM   1314  CB  MET A 168      29.962 -13.813  23.722  1.00 46.58           C  \nATOM   1315  CG  MET A 168      31.310 -13.538  23.144  1.00 49.54           C  \nATOM   1316  SD  MET A 168      32.251 -12.652  24.414  1.00 52.83           S  \nATOM   1317  CE  MET A 168      33.539 -12.297  23.270  1.00 46.37           C  \nATOM   1318  N   LYS A 169      29.807 -17.014  23.574  1.00 48.84           N  \nATOM   1319  CA  LYS A 169      30.531 -18.288  23.386  1.00 45.38           C  \nATOM   1320  C   LYS A 169      29.730 -19.240  22.523  1.00 47.76           C  \nATOM   1321  O   LYS A 169      30.311 -19.874  21.653  1.00 45.76           O  \nATOM   1322  CB  LYS A 169      30.904 -18.909  24.733  1.00 47.14           C  \nATOM   1323  CG  LYS A 169      32.043 -18.140  25.410  1.00 45.73           C  \nATOM   1324  CD  LYS A 169      32.532 -18.824  26.692  1.00 46.77           C  \nATOM   1325  CE  LYS A 169      31.537 -18.474  27.777  1.00 48.65           C  \nATOM   1326  NZ  LYS A 169      31.737 -19.305  28.982  1.00 55.73           N  \nATOM   1327  N   GLU A 170      28.405 -19.272  22.713  1.00 40.10           N  \nATOM   1328  CA  GLU A 170      27.523 -19.983  21.801  1.00 43.26           C  \nATOM   1329  C   GLU A 170      27.697 -19.538  20.363  1.00 56.52           C  \nATOM   1330  O   GLU A 170      27.732 -20.369  19.501  1.00 58.93           O  \nATOM   1331  CB  GLU A 170      26.073 -19.829  22.197  1.00 37.52           C  \nATOM   1332  CG  GLU A 170      25.786 -20.262  23.635  1.00 42.33           C  \nATOM   1333  CD  GLU A 170      26.125 -21.730  23.900  1.00 44.18           C  \nATOM   1334  OE1 GLU A 170      26.490 -22.443  22.930  1.00 43.17           O  \nATOM   1335  OE2 GLU A 170      25.973 -22.174  25.066  1.00 44.36           O  \nATOM   1336  N   LYS A 171      27.810 -18.232  20.094  1.00 58.07           N  \nATOM   1337  CA  LYS A 171      27.973 -17.779  18.705  1.00 59.82           C  \nATOM   1338  C   LYS A 171      29.347 -18.099  18.151  1.00 60.77           C  \nATOM   1339  O   LYS A 171      29.594 -17.777  17.013  1.00 61.46           O  \nATOM   1340  CB  LYS A 171      27.757 -16.258  18.530  1.00 52.76           C  \nATOM   1341  CG  LYS A 171      26.486 -15.701  19.083  1.00 55.01           C  \nATOM   1342  CD  LYS A 171      25.296 -16.110  18.294  1.00 62.99           C  \nATOM   1343  CE  LYS A 171      24.191 -15.168  18.724  1.00 70.79           C  \nATOM   1344  NZ  LYS A 171      22.922 -15.752  18.244  1.00 80.15           N  \nATOM   1345  N   GLY A 172      30.254 -18.697  18.926  1.00 55.80           N  \nATOM   1346  CA  GLY A 172      31.588 -18.967  18.389  1.00 58.64           C  \nATOM   1347  C   GLY A 172      32.478 -17.723  18.363  1.00 72.32           C  \nATOM   1348  O   GLY A 172      33.589 -17.757  17.807  1.00 76.83           O  \nATOM   1349  N   VAL A 173      32.009 -16.619  18.972  1.00 70.42           N  \nATOM   1350  CA  VAL A 173      32.871 -15.449  19.200  1.00 66.95           C  \nATOM   1351  C   VAL A 173      33.793 -15.658  20.445  1.00 70.38           C  \nATOM   1352  O   VAL A 173      33.304 -15.771  21.562  1.00 79.08           O  \nATOM   1353  CB  VAL A 173      32.070 -14.088  19.235  1.00 67.10           C  \nATOM   1354  CG1 VAL A 173      33.063 -12.933  19.294  1.00 56.04           C  \nATOM   1355  CG2 VAL A 173      31.092 -13.912  18.035  1.00 51.53           C  \nATOM   1356  N   LYS A 174      35.117 -15.703  20.239  1.00 82.69           N  \nATOM   1357  CA  LYS A 174      36.089 -16.117  21.285  1.00 78.41           C  \nATOM   1358  C   LYS A 174      36.337 -15.015  22.285  1.00 75.74           C  \nATOM   1359  O   LYS A 174      36.922 -13.970  21.937  1.00 68.16           O  \nATOM   1360  CB  LYS A 174      37.452 -16.646  20.726  1.00 74.31           C  \nATOM   1361  CG  LYS A 174      37.346 -17.888  19.840  1.00 79.46           C  \nATOM   1362  CD  LYS A 174      36.116 -18.714  20.238  1.00 78.71           C  \nATOM   1363  CE  LYS A 174      35.741 -19.728  19.182  1.00 83.20           C  \nATOM   1364  NZ  LYS A 174      36.758 -20.805  19.211  1.00 94.72           N  \nATOM   1365  N   PRO A 175      35.905 -15.242  23.543  1.00 65.34           N  \nATOM   1366  CA  PRO A 175      36.213 -14.261  24.573  1.00 65.97           C  \nATOM   1367  C   PRO A 175      37.694 -13.941  24.597  1.00 60.59           C  \nATOM   1368  O   PRO A 175      38.520 -14.866  24.464  1.00 62.20           O  \nATOM   1369  CB  PRO A 175      35.793 -14.989  25.840  1.00 62.06           C  \nATOM   1370  CG  PRO A 175      34.561 -15.708  25.387  1.00 64.95           C  \nATOM   1371  CD  PRO A 175      34.974 -16.270  24.048  1.00 70.80           C  \nTER    1372      PRO A 175                                                      \nHETATM 1373  O   HOH A 201      40.280  20.781   3.244  1.00 52.84           O  \nHETATM 1374  O   HOH A 202      20.293   6.181  19.962  1.00 64.29           O  \nHETATM 1375  O   HOH A 203      29.101  16.768  15.916  1.00 35.91           O  \nHETATM 1376  O   HOH A 204      30.321   8.355  24.707  1.00 68.78           O  \nHETATM 1377  O   HOH A 205      23.840  -0.738   1.226  1.00 61.85           O  \nHETATM 1378  O   HOH A 206      38.411  21.850  -4.532  1.00 59.33           O  \nHETATM 1379  O   HOH A 207      20.542  11.097  20.391  1.00 57.82           O  \nHETATM 1380  O   HOH A 208      49.727  19.944 -10.408  1.00 41.29           O  \nHETATM 1381  O   HOH A 209      18.181   2.886  15.484  1.00 54.91           O  \nHETATM 1382  O   HOH A 210      55.365  16.066  -0.232  1.00 43.20           O  \nHETATM 1383  O   HOH A 211      32.580  -2.211   5.747  1.00 64.58           O  \nHETATM 1384  O   HOH A 212      24.500  15.099   6.701  1.00 48.36           O  \nHETATM 1385  O   HOH A 213      54.101  17.066  -7.910  1.00 58.74           O  \nHETATM 1386  O   HOH A 214      28.906  18.593   9.807  1.00 53.68           O  \nHETATM 1387  O   HOH A 215      36.347  21.690  -1.610  1.00 61.24           O  \nHETATM 1388  O   HOH A 216      21.519  12.286  16.589  1.00 53.99           O  \nHETATM 1389  O   HOH A 217      25.752  -9.874   9.181  1.00 76.83           O  \nHETATM 1390  O   HOH A 218      24.956  17.803   8.168  1.00 61.21           O  \nHETATM 1391  O   HOH A 219      42.284   7.443   1.027  1.00 70.35           O  \nHETATM 1392  O   HOH A 220      37.413   7.314  10.370  1.00 66.88           O  \nHETATM 1393  O   HOH A 221      30.744  20.597  12.223  1.00 44.12           O  \nHETATM 1394  O   HOH A 222      27.559  18.259   8.067  1.00 51.07           O  \nHETATM 1395  O   HOH A 223      24.291 -15.896  37.527  1.00 47.27           O  \nHETATM 1396  O   HOH A 224      39.268  19.437   9.470  1.00 48.79           O  \nHETATM 1397  O   HOH A 225      23.122 -16.401  27.726  1.00 53.12           O  \nHETATM 1398  O   HOH A 226      22.061  -7.264  35.178  1.00 50.24           O  \nHETATM 1399  O   HOH A 227      19.930  11.588  14.940  1.00 61.89           O  \nHETATM 1400  O   HOH A 228      20.818   4.793  26.571  1.00 69.16           O  \nHETATM 1401  O   HOH A 229      45.946  21.822  -0.607  1.00 56.88           O  \nHETATM 1402  O   HOH A 230      33.807   0.379   6.057  1.00 60.90           O  \nHETATM 1403  O   HOH A 231      23.626  -7.737  38.240  1.00 65.60           O  \nHETATM 1404  O   HOH A 232      47.008  16.135   7.243  1.00 60.75           O  \nHETATM 1405  O   HOH A 233      21.129   3.640  29.662  1.00 70.32           O  \nHETATM 1406  O   HOH A 234      29.979 -21.702  30.648  1.00 58.46           O  \nHETATM 1407  O   HOH A 235      23.845 -18.238  25.780  1.00 45.06           O  \nHETATM 1408  O   HOH A 236      41.470  19.276  13.194  1.00 65.46           O  \nHETATM 1409  O   HOH A 237      32.862  -0.186  26.013  1.00 72.46           O  \nHETATM 1410  O   HOH A 238      39.126  19.346  12.234  1.00 58.14           O  \nHETATM 1411  O   HOH A 239      50.045  21.320  -8.411  1.00 57.54           O  \nHETATM 1412  O   HOH A 240      37.058  27.171  -8.141  1.00 69.03           O  \nHETATM 1413  O   HOH A 241      36.183  -2.886  25.111  1.00 64.90           O  \nHETATM 1414  O   HOH A 242      24.805 -16.637  32.501  1.00 57.86           O  \nHETATM 1415  O   HOH A 243      21.693  -6.986  37.878  1.00 58.61           O  \nHETATM 1416  O   HOH A 244      55.783  14.880   1.979  1.00 68.15           O  \nHETATM 1417  O   HOH A 245      31.327  19.387  19.252  1.00 57.40           O  \nMASTER      334    0    0   10    0    0    0    6 1410    1    0   14          \nEND                                                                             ","format":"pdb"},{"method":"setStyle","style":{"cartoon":{"color":"spectrum","style":"rectangle","ribbon":false,"arrows":true,"tubes":false,"thickness":0.4,"opacity":1}},"sel":{}},{"method":"addStyle","style":{"stick":{"hidden":false,"radius":0.3,"singleBonds":false,"colorscheme":"default","opacity":1},"sphere":{"hidden":false,"scale":0.3,"colorscheme":"default","opacity":1}},"sel":{"invert":true}},{"method":"zoomTo"},{"method":"spin","axis":"y","speed":0},{"method":"setViewStyle","style":{"style":"outline","color":"black","width":0.1},"width":0.1,"color":"black"},{"method":"zoomTo","sel":{}},{"method":"spin","axis":"y","speed":1}]},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<p>Check the Twitter thread for more details, but now while still in the Rstudio IDE, you can create molecular visualisations to quickly preview your work, but more importantly, you can seamlessly embed them into <em>Rmarkdown</em> HTML reports for sharing with colleagues or blogging online.</p>
<blockquote class="twitter-tweet blockquote">
<p lang="en" dir="ltr">
Hey! I helped make an open source package!<br><br>If you do any kind of structural bioinformatics in R, or even if you just want to embed pdbs easily into a blogdown post, you can do so with the {r3dmol} (originally by <a href="https://twitter.com/swsoyee?ref_src=twsrc%5Etfw"><span class="citation" data-cites="swsoyee">@swsoyee</span></a>): <a href="https://t.co/s9De6kuVpT">https://t.co/s9De6kuVpT</a> <a href="https://twitter.com/hashtag/rstats?src=hash&amp;ref_src=twsrc%5Etfw">#rstats</a> <a href="https://t.co/LA9rjJIaKV">pic.twitter.com/LA9rjJIaKV</a>
</p>
— Brady Johnston (<span class="citation" data-cites="bradyajohnston">@bradyajohnston</span>) <a href="https://twitter.com/bradyajohnston/status/1357476352010510341?ref_src=twsrc%5Etfw">February 4, 2021</a>
</blockquote>
<script async="" src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>


</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>